Content: New Hai advanced start (#7721)
endless-sky/endless-sky: data/starts.txt

		has "event: hai-human resolution announced"
	conversation "hai intro"
	account
		credits 1431000
		score 400
		mortgage Mortgage
			principal 1431000
			interest 0.003
			term 730
	set "hai start"
	# need to set it here to not have it trigger alongside the invisible mission
	set "First Contact: Hai: offered"
	set "Intro [0]: offered"

conversation "hai intro"
	scene "scene/hai shipyard"
	`	You've never been to the shipyard before, and going to the poles already feels like you're traveling to another world. The pictures and videos you've seen of all these ships before simply don't do them justice; seeing them for real is something else entirely. Even though you know a lot of those ships must have been in service for many years, you are stunned by how new and shiny they look. You begin to wonder if you are ready to take the big leap.`
	`	This "Aphid" could suit your needs. It is fast and able to carry a respectable amount of cargo.`
	scene "thumbnail/hai aphid"
	`	That is, unless you want to get adventurous and try out the impetuous Grasshopper. Surely the interceptor would be able to beat the smaller human warships, provided a competent pilot is at the helm.`

> 	`	You have, after all, heard tales of pirates and quick money from the many newcomers. Thanks to the installation of Mountaintop, they say, the way to human space is much safer than it used to be. What happened with the Pug is not clear, however.`

Reviewer: "Mountaintop" may need some explanation, I think. Even though the player has played the game before, they might be coming back to it after a while. They may also just not remember what that is.

Author: this should no longer be here bc we're not skipping HR

	`	Alternatively, up north you can find the 'Unfettered'. Maybe they would come to like you, if you were to show them the strength and cunning they respect.`
	choice
		`	(Go to the bank.)`
	scene "scene/hai bank"
	`	You head to the bank's lobby, hoping to get enough of a loan to be able to buy one of these pristine ships.`
	`	To your disappointment, the only ship you could afford is an Aphid. You arrange the contract to loan as much money as you need in order to buy one, in what has to be the first seriously unwise act of your life.`

> 	`	You've heard of new captains stripping down their ships to recoup the initial buy-in, which is crossing your mind at seeing the financial burden you're about to take on. While waiting for paperwork to be processed, you scan over an Aphid pamphlet in the office. The pricey shield regenerator could be traded in if dangerous areas are avoided... and perhaps even the anti-missile system could be downgraded or outright removed... one of the Sand Cells could be sold back to make more space for cargo...`

Reviewer: ```suggestion
	`	You've heard of new captains stripping down their ships to recoup the initial buy-in, which is crossing your mind as you see the financial burden you're about to take on. While waiting for paperwork to be processed, you scan over an Aphid pamphlet in the office. The pricey shield regenerator could be traded in if dangerous areas are avoided... and perhaps even the anti-missile system could be downgraded or outright removed... one of the Sand Cells could be sold back to make more space for cargo...`
```

> 	`	Those anxious musings are interrupted when the bank employee finally lays it out to you, with a hesitant smile. "This will put you in a debt of 1.431 million credits, due in the course of 2 years, with an interest rate of 0.3% per day. Please sign here."`

Reviewer: ```suggestion
	`	Those anxious musings are interrupted when the bank employee finally lays it out to you, with a hesitant smile. "You are accepting a loan in the amount of 1,431,000 credits, due over the course of 2 years, with an interest rate of 0.3% per day. Please sign here."`
```
A banker would be price, I think.

#
# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
# PARTICULAR PURPOSE. See the GNU General Public License for more details.
#
# You should have received a copy of the GNU General Public License along with
# this program. If not, see <https://www.gnu.org/licenses/>.

start "default"
	name "Endless Sky"
	description `You grew up on New Boston, an uninteresting world in the Dirt Belt. You've dreamed of owning a starship ever since your first visit to the spaceport at the age of fifteen. After many long years of working at a textile mill, you've finally saved enough credits to apply for a pilot's license from the Republic.`
	description `	After receiving the license and with freedom from this planet at your fingertips, you travel to the bank to take out a loan on your first ship...`
	thumbnail "scene/lobby"
	date 16 11 3013
	system "Rutilicus"
	planet "New Boston"
	conversation "default intro"
	account
		credits 480000
		score 400
		mortgage Mortgage
			principal 480000
			interest 0.004
			term 365

> 	set "license: Pilot's"

Reviewer 2: Use a "human space start" condition for all starts in human space. That way we can check `not "human space start"` in condition sets.
```suggestion
	set "license: Pilot's"
	set "species: human"
	set "human space start"
```

Author: I didnt do this bc it wouldnt apply retroactively to saves, but we could just add a compatibility mission?

Reviewer 2: Well the more starts we add, the more convoluted the conditions would get if we didn't set it up this way. If a compatibility mission is what we need, then it's what we'll do.

Author: makes sense, I did it already

	`	The only moment when he becomes truly animated is when explaining the Truth in Lending statement to you. "You are borrowing 480,000 credits," he says, "to be repaid over the course of one year. Your daily interest rate is 0.4%, which means that your daily payments are 2,503 credits, and by the end of the year you will have paid... 433,567 credits in interest. Sign here." He grins.`
	`	You sign your name one final time:`
	name
	`	The elevator is so well-tuned that you do not even realize it is moving until it has deposited you back in the lobby. But as you leave the bank, you are smiling. This crazy adventure suddenly feels real to you. You are going to do it. You are finally going to get off this planet.`
	`	Compared to the bank, you feel much more at home in the shipyard, walking among the rusted out hulks and newer ships that gleam in the sunlight. You smell grease and dirt and rocket fuel; wonderful smells. There are three ship models within your price range. Which one you choose will determine your future...`

start "hai space"
	name "New Hai"

> 	description `You grew up on Greenwater, a world with a very rich and mixed culture. You've dreamed of owning your own fleet since you were given toy spaceships as a small child. In order to reach that goal, you worked hard making human clothing that sold well.`

Reviewer 2: Think of something, anything, other than first last being just another textile worker (or textile-adjacent) in just another location. This is basically just a repeat of #6352.

Author: in this case its making human clothing which is a human expertise and a thing I imagined the hai would like but mb you can be a sales manager? I'll think of smth but that may mean more references to tweak

Reviewer 2: That seems distinct enough to me.

	`	The only moment when he becomes truly animated is when explaining the Truth in Lending statement to you. "You are borrowing 480,000 credits," he says, "to be repaid over the course of one year. Your daily interest rate is 0.4%, which means that your daily payments are 2,503 credits, and by the end of the year you will have paid... 433,567 credits in interest. Sign here." He grins.`
	`	You sign your name one final time:`
	name
	`	The elevator is so well-tuned that you do not even realize it is moving until it has deposited you back in the lobby. But as you leave the bank, you are smiling. This crazy adventure suddenly feels real to you. You are going to do it. You are finally going to get off this planet.`
	`	Compared to the bank, you feel much more at home in the shipyard, walking among the rusted out hulks and newer ships that gleam in the sunlight. You smell grease and dirt and rocket fuel; wonderful smells. There are three ship models within your price range. Which one you choose will determine your future...`

start "hai space"
	name "Hai Space Start"

> 	description `You grew up on Greenwater, a world with a very rich and mixed culture. You've dreamed of owning your own fleet since you were given toy spaceships as a small child. In order to reach that goal, you worked hard at a travel company, recommending mostly locations in human space.`

Reviewer 3: ```suggestion
	description `You grew up on Greenwater, a world with a very rich and mixed culture. You've dreamed of owning your own fleet since you were given toy spaceships as a small child. In order to reach that goal, you worked hard at a travel company, mostly recommending locations in human space.`
```

		credits "over a million"
	to reveal
		has "main plot completed"
		has "Ask the Hai about the Unfettered"
	on reveal
		name "Undiscovered Hai Start"
		description "There could be interesting possibilities up north, once all crises are resolved."
		description "	You could have started a different way, went a different path."
		system "Somewhere far north"
		planet "Mixed culture"
		date "Same old one"
		credits 1.431M
		debt 1.431M
	to unlock
		has "main plot completed"
		has "event: hai-human resolution announced"
	conversation "hai intro"
	account
		credits 1431000
		score 400
		mortgage Mortgage
			principal 1431000
			interest 0.003
			term 730

> 	set "hai start"
> 	set "non-human space start"

Reviewer 2: Let's not confuse "hai start" with "starting as a Hai." Let's rename "hai start" to "hai space start", then clarify that you're still a human with a "species: human" condition. I think we can also drop the "non-human space start" condition given my next suggestion.
```suggestion
	set "species: human"
	set "hai space start"
```

	set "First Contact: Hai: offered"
	set "Intro [0]: offered"

conversation "hai intro"
	scene "scene/hai shipyard"
	`Going to the North Pole already makes you feel as if you're on another world. Though you've met some Hai in the past, you've never been among so many of them in one place; the movement of the crowd in this Hai city far outstrips the equatorial human town where you grew up.`
	scene "scene/hai bank"
	`	You head to the bank's lobby, hoping to get enough of a loan to be able to buy one of the pristine ships advertised here.`

> 	`	To your disappointment, the only ship you can afford is an Aphid. You arrange the contract to loan as much money as you need in order to buy one, in what has to be the first seriously unwise act of your life. However, upon receiving the papers you find out they are in Hai, which you can't make sense of. Probably seeing the confusion on your face, they promptly give you new papers and give you time to read. Only now do you fully realize how risky this is, you might have to cut some costs once you get your hands on the ship.`

Reviewer 3: ```suggestion
	`	To your disappointment, the only ship you can afford is an Aphid. You arrange the contract to loan as much money as you need in order to buy one, in what has to be the first seriously unwise act of your life. However, upon receiving the papers you find out they are in Hai, which you can't make sense of. Probably seeing the confusion on your face, they promptly give you new papers and give you time to read. Only now do you fully realize how risky this is. You might have to cut some costs once you get your hands on the ship.`
```

	`	Those anxious musings are interrupted when the bank employee finally lays it out to you, with a hesitant smile. "You are accepting a loan in the amount of 1,431,000 credits, due over the course of 2 years, with an interest rate of 0.3% per day. Please sign here."`
	name
	`	You head out of the bank, but see a notice on your way to buy the Aphid at the shipyard. A retiring captain is selling off his fleet at a reduced price, and one of the ships he's offering is a Grasshopper... maybe you could get a deal on it.`
	choice
		`	(Go to the shipyard.)`
			goto shipyard
		`	(Go see what the captain has to offer first.)`

> 	`	You head to the spaceport and easily find the captain, a Hai, selling one of his Centipedes to another Hai. Thankfully it looks like he has not sold the Grasshopper yet. They happen to be conversing in your language, and you can overhear part of the conversation as the buyer is speaking. "... great that some people still care about helping others make their way to the stars faster, instead of only looking at profit."`

Reviewer 3: Looks like I missed something. 
```suggestion
	`	You head to the spaceport and easily find the captain, who is a Hai, selling one of his Centipedes to another Hai. Thankfully, it looks like he has not sold the Grasshopper yet. They happen to be conversing in your language, and you can overhear part of the conversation as the buyer is speaking. "... great that some people still care about helping others make their way to the stars faster, instead of only looking at profit."`
```

	`	"What a noble goal indeed," she says, before noticing you, "But it seems you have a new customer here. I will leave you to it." The captain turns to you, "Hello, good fellow! I assume you come for the Grasshopper."`
	choice
		`	"Indeed."`
		`	"How did you know?"`
		`	"Sorry, not interested."`
			decline
	`	"I can see the will to explore in your eyes, and with your age it just seems like the perfect fit. I would be delighted to have you flying the 'Pristine Jewel'!" he says, taking you to the ship. It is clearly not brand new, but is still in a surprisingly good condition.`
	scene "thumbnail/hai grasshopper"
	`	"Because I like you I'll give you a friendly price. What do you say about... a bit over a million credits, for this wonderful ship? That's less than a third the price you would have to pay if you were to buy this model from the shipyard. Except you can't find a model like this in shipyards, trust me!"`
	choice
		`	"Sold!"`
		`	"Sorry, not interested. I'll go to the shipyard instead."`
			goto shipyard

> 	`	As the two of you settle up the paperwork, the captain takes the time to explain, in great detail, why this ship is so marvelous. In your excitement to buy the ship all you can hear from the Hai captain's speech is the repeated reference to the Grasshopper's 'exceptional' reinforced armor.`

Reviewer 3: ```suggestion
	`	As the two of you settle up the paperwork, the captain takes the time to explain, in great detail, why this ship is so marvelous. In your excitement to buy the ship, all you can hear from the Hai captain's speech is the repeated reference to the Grasshopper's 'exceptional' reinforced armor.`
```

	action
		set "hai space start: Grasshopper"
		give ship "Grasshopper (Expensive)" "Pristine Jewel"
		payment -1132000
	`	Before you leave, he gives you a final tip: "If you want to, you can always sell this ship and buy an Aphid instead. The shipyard authorities will give you a discount for having sold my ship."`
		decline
	label shipyard

> 	`	The pictures and videos you've seen of Hai ships before simply don't do them justice; seeing them in all their splendor here in the shipyard is something else entirely. Even though you know a lot of those ships must have been in service for many years, you are stunned by how new and shiny they look.`

Reviewer 3: The player’s never seen a Hai ship near the equator before?

Author: Yeah they've never been to the shipyards

Reviewer 3: But Hai ships would have landed at the town they live at, right?

Author: I don't think so, you'd need a spaceport and it isnt that big
this planet has humans living near the equator and hai at the poles

Reviewer 3: OK. This is resolved than.

# this ship uses the most expensive loadout possible (3.5M)
# so that you +- have the same amount of value back if you sell it (1.3mill)
# It has a small hull buff to help the player survive.

> ship "Grasshopper" "Grasshopper (Expensive)"

Reviewer 3: Expensive might want to be used elsewhere. 
```suggestion
ship "Grasshopper" "Grasshopper (Hai Start)"
```

	add attributes
		"hull" 400
	outfits
		`"Benga" Atomic Thruster`
		`"Biroo" Atomic Steering`
		"Fuel Pod"
		"Hai Chasm Batteries"
		"Hai Corundum Regenerator"
		"Hai Williwaw Cooling"
		Hyperdrive
		"Ion Cannon"
		"Pebble Core"
		"Pulse Rifle"
		"Quantum Keystone" 2

# the start itself cannot initiate events
mission "Greenwater start"
	landing
	invisible
	to offer
		has "hai space start"
	on offer
		event "not from boston" 0
		event "grew up on greenwater" 0
		log "Finally scraped together enough money for a down payment on a starship. The interest on the mortgage is exorbitant."

> 		log "Factions" "Republic" `Hundreds of years ago, the independent territories in different parts of human space agreed to join together into a single democratic government, with Earth as its capital. The rise of the Republic ushered in a long period of peace and prosperity in human history.`

Reviewer 3: Does the player know enough about the Republic to make entries in their log? Maybe they were born to human parents in Hai space. 
It would be kind of cool to have a `First Contact: Republic`

Author: The planet you start on has a lot of humans on it, and the hai would definitely know about the republic

Reviewer 3: Okay.

		log "Factions" "Syndicate" `The Syndicate is a megacorporation, the largest employer in human space. People who cannot find steady work elsewhere flock to the Syndicate factory worlds, where they are almost guaranteed to find a job. Although the Syndicate is technically part of the Republic, Syndicate worlds are governed directly by the corporation.`
			`The Syndicate is a central part of the Republic's economy, but they are also well known for selling shoddy products and for doing damage to the environment on the worlds they control. Their treatment of workers is questionable, and their opposition to organized labor is legendary. Some Syndicate factory towns have even been accused of human rights abuses.`
		log "Factions" "Pirates" `In poorer and more remote star systems, where the Navy seldom patrols, pirate attacks on merchant ships are frequent. Pirates are also known to attack large, unguarded convoys of freighters even in more populated areas. Most pirate fleets come from lawless worlds on the outskirts of human space.`

> 		log "Factions" "Hai" `The Hai are a species of giant, intelligent rodents, who live to the north of human space. They allow any humans who discover their territory to live alongside them, and to trade with them and purchase their technology.`

Reviewer 3: They don’t have an entry about the Unfettered?

	`	As the two of you settle up the paperwork, the captain takes the time to explain, in great detail, why this ship is so marvelous. In your excitement to buy the ship, all you can hear from the Hai captain's speech is the repeated reference to the Grasshopper's 'exceptional' reinforced armor.`
	action
		set "hai space start: Grasshopper"
		give ship "Grasshopper (Hai Start)" "Pristine Jewel"
		payment -1132000
	`	Before you leave, he gives you a final tip: "If you want to, you can always sell this ship and buy an Aphid instead. The shipyard authorities will give you a discount for having sold my ship."`
		decline
	label shipyard
	`	The pictures and videos you've seen of Hai ships before simply don't do them justice; seeing them in all their splendor here in the shipyard is something else entirely. Even though you know a lot of those ships must have been in service for many years, you are stunned by how new and shiny they look.`

# this ship uses the most expensive loadout possible (3.5M)
# so that you +- have the same amount of value back if you sell it (1.3mill)
# It has a small hull buff to help the player survive.
ship "Grasshopper" "Grasshopper (Hai Start)"
	add attributes

> 		"hull" 400

Author: Northern human space is much more dangerous than the starting human location. And then there's Wah Ki that's dangerous as well (the risks are just much higher, you can't just fight pirates like in human space)

This is to make it a bit more viable to migrate there with this ship eventually (or at the start), and make the Grasshopper have some kind of an upside over the Aphid, so it's not just an autopick.
If you worry about the balance side I'm happy with this being only disabled hull hp as the point of it is just so you don't get killed all the time (but like half of it already is I think)

> 		"hull" 400

Reviewer 4: Northern space below Alnitak isn't that dangerous; you can reliably travel the Heia Due-Betelgeuse route without seeing even a single pirate.

The Grasshopper already has a lot of advantages over using an Aphid. You lose far less credits selling after full depreciation (260k loss on the Grasshopper versus ~700k on a stripped down Aphid). You're also getting more relevant stats when paying the Hai premium on the Grasshopper; the extra health, speed and regeneration from Hai hulls and outfits is always going to help you on an interceptor. What you get by paying extra on the Aphid is 51 cargo space, going up to 141 space with cargo expansions. The Star Barge, whose hull costs 380k less, gets 50 cargo space, going up to 110 with expansions. There's also less competition on the Grasshopper's end: all the other combat ships available at 1.4 million are interceptors with worse stats than the Grasshopper, while the Aphid has to compete with ships like the Scout, the Bounder, the Clipper and the Freighter.

> 		"hull" 400

Author: You'll still end up in a dangerous area of space where Leviathans are spawning and you only have a bit of fuel, but ok I removed the buff

	`	The only moment when he becomes truly animated is when explaining the Truth in Lending statement to you. "You are borrowing 480,000 credits," he says, "to be repaid over the course of one year. Your daily interest rate is 0.4%, which means that your daily payments are 2,503 credits, and by the end of the year you will have paid... 433,567 credits in interest. Sign here." He grins.`
	`	You sign your name one final time:`
	name
	`	The elevator is so well-tuned that you do not even realize it is moving until it has deposited you back in the lobby. But as you leave the bank, you are smiling. This crazy adventure suddenly feels real to you. You are going to do it. You are finally going to get off this planet.`
	`	Compared to the bank, you feel much more at home in the shipyard, walking among the rusted out hulks and newer ships that gleam in the sunlight. You smell grease and dirt and rocket fuel; wonderful smells. There are three ship models within your price range. Which one you choose will determine your future...`

start "hai space"
	name "New Hai"

> 	description `You grew up on Greenwater, a world with a very rich and mixed culture. You've dreamed of owning your own fleet since you were given toy spaceships as a small child. In order to reach that goal, you worked hard at a travel company, recommending mostly locations in human space.`

Author: it aint a mission so pretty sure it wont work, looking at the code

	date 16 11 3013
	to display
		or
			has "main plot completed"
			has "Ask the Hai about the Unfettered"
	on display
		description "What if you had not grown up on New Boston? What if you had waited longer before taking to the skies?"
		thumbnail ""
		date "3020"
		planet "Slightly too warm"
		credits "over a million"
	to reveal
		has "main plot completed"
		has "Ask the Hai about the Unfettered"
	on reveal
		name "Undiscovered Hai Start"
		description "There could be interesting possibilities up north, once all crises are resolved."
		description "	You could have started a different way, went a different path."
		system "Somewhere far north"
		planet "Mixed culture"
		date "Same old one"
		credits 1.431M
		debt 1.431M
	to unlock
		has "main plot completed"
		has "event: hai-human resolution announced"
	conversation "hai intro"
	account
		credits 1431000
		score 400
		mortgage Mortgage
			principal 1431000
			interest 0.003
			term 730
	set "species: human"
	set "hai space start"
	# need to set it here to not have it trigger alongside the invisible mission
	set "First Contact: Hai: offered"
	set "Intro [0]: offered"

conversation "hai intro"
	scene "scene/hai shipyard"

> 	`	You've never been to the shipyard before, and going to the North Pole already feels like you're traveling to another world. The pictures and videos you've seen of all these ships before simply don't do them justice; seeing them for real is something else entirely. Even though you know a lot of those ships must have been in service for many years, you are stunned by how new and shiny they look. You begin to wonder if you are ready to take the big leap.`

Reviewer 5: ```suggestion
	`You've never been to the shipyard before, and going to the North Pole already feels like you're traveling to another world. The pictures and videos you've seen of all these ships before simply don't do them justice; seeing them for real is something else entirely. Even though you know a lot of those ships must have been in service for many years, you are stunned by how new and shiny they look. You begin to wonder if you are ready to take the big leap.`
```

	`	This "Aphid" could suit your needs. It is fast and able to carry a respectable amount of cargo.`
	scene "thumbnail/hai aphid"
	`	That is, unless you want to get adventurous and try out the impetuous Grasshopper. Surely the interceptor would be able to beat the smaller human warships, provided a competent pilot is at the helm.`

> 	`	You have, after all, heard tales of pirates and quick money from the many newcomers to Hai space, for those willing to take big risks, that is.`

Reviewer 6: ```suggestion
	`	You have, after all, heard tales of pirates and quick money from the many newcomers to Hai space - for those willing to take big risks, that is.`
```

	`	Alternatively, up north you can find the 'Unfettered'. Maybe they would come to like you, if you were to show them the strength and cunning they respect.`
	choice
		`	(Go to the bank.)`
	scene "scene/hai bank"
	`	You head to the bank's lobby, hoping to get enough of a loan to be able to buy one of these pristine ships.`
	`	To your disappointment, the only ship you could afford is an Aphid. You arrange the contract to loan as much money as you need in order to buy one, in what has to be the first seriously unwise act of your life.`

> 	`	You've heard of new captains stripping down their ships to recoup the initial buy-in, which is crossing your mind as you see the financial burden you're about to take on. While waiting for paperwork to be processed, you scan over an Aphid pamphlet in the office. The pricey shield regenerator could be traded in if dangerous areas are avoided... and perhaps even the anti-missile system could be downgraded or outright removed... one of the Sand Cells could be sold back to make more space for cargo...`

Reviewer 5: ```suggestion
	`	You've heard of new captains stripping down their ships to recoup the initial buy-in, which is crossing your mind as you see the financial burden you're about to take on. As you wait for the paperwork to be processed, you passively scan over an Aphid pamphlet in the office. The pricey shield regenerator could be traded in if dangerous areas are avoided... and perhaps even the anti-missile system could be downgraded or outright removed... one of the Sand Cells could be sold back to make more space for cargo...`
```

	`	Those anxious musings are interrupted when the bank employee finally lays it out to you, with a hesitant smile. "You are accepting a loan in the amount of 1,431,000 credits, due over the course of 2 years, with an interest rate of 0.3% per day. Please sign here."`
	name
	`	You head out of the bank, but on your way to buy the Aphid at the shipyard, you see a notice. A retiring captain is offering to sell some of his ships at an interesting price, including a Grasshopper... Maybe they could sell it to you.`
	choice
		`	(Go to the shipyard.)`
			decline
		`	(Go see what the captain has to offer first.)`

> 	`	You head to the spaceport and easily find the captain selling off his ships. Thankfully they did not yet sell the Grasshopper.`

Reviewer 6: ```suggestion
	`	You head to the spaceport and easily find the captain selling off his ships. Thankfully he has not yet sold the Grasshopper.`
```

	`	The captain greets you in Hai. You are embarrassed to have to explain to them that you never learned their language. Most Hai speak your language however, and, as usual, they are able to switch to speaking yours without much difficulty.`
	`	"Hello, good fellow! I assume you come for the Grasshopper." Your interest for this ship must be apparent.`
	choice
		`	"Yes indeed."`
		`	"How did you know?"`
		`	"Sorry, not interested."`
			decline
	`	"I can see the will to explore in your eyes, but it is not only that: I know that some humans want to head back to their space, now that it is safer to do so. I would be delighted to have you flying the 'Pristine Jewel'!" they say, taking you to the ship. It is clearly not brand new, but is still in a surprisingly good condition.`
	scene "thumbnail/hai grasshopper"
	`	"Because I like you I'll give you a friendly price. What do you say about... a bit over a million credits, for this wonderful ship? That's less than a third the price you would have to pay if you were to buy this model from the shipyard. Except you can't find a model like this in shipyards, trust me!"`
	choice
		`	"Sold!"`
		`	"Sorry, not interested."`
			decline

> 	`	As the two of you settle up the paperwork, the captain takes the time to explain, in great detail, why this ship is so marvelous. Because of how excited you are to finally fly, you miss everything other than repeated reference to the 'exceptional' reinforced armor.`

Reviewer 5: > Because of how excited you are to finally fly, you miss everything other than repeated reference to the 'exceptional' reinforced armor.

That feels a little contrived. How about this:
```suggestion
	`	As the two of you settle up the paperwork, the captain takes the time to explain, in great detail, why this ship is so marvelous. In your excitement to buy the ship all you can hear from the Hai captain's speech is the repeated reference to the Grasshopper's 'exceptional' reinforced armor.`
```

	`	Before you leave, they give you a final tip: "If you want to, you can always sell this ship and buy an Aphid instead. The shipyard authorities will give you a discount for having sold my ship."`
	action
		set "hai space start: Grasshopper"
		give ship "Grasshopper (Expensive)" "Pristine Jewel"
		payment -1132000

# this ship uses the most expensive loadout possible (3.5M)
# so that you +- have the same amount of value back if you sell it (1.3mill)

> # It has a small hull buff to help the player survive.

Reviewer 5: We don't really give this treatment to other ships, but might be interesting.

Author: Main reason I did this is bc the player might be fighting the uhai a bit, or the pirates near the wormhole; enemies in the neighborhood are much tougher than your average pirate in paradise space

I def think it'd be nice to do this for a few others yeah (mb up the disabled ship hp too)

		log "Factions" "Syndicate" `The Syndicate is a megacorporation, the largest employer in human space. People who cannot find steady work elsewhere flock to the Syndicate factory worlds, where they are almost guaranteed to find a job. Although the Syndicate is technically part of the Republic, Syndicate worlds are governed directly by the corporation.`
			`The Syndicate is a central part of the Republic's economy, but they are also well known for selling shoddy products and for doing damage to the environment on the worlds they control. Their treatment of workers is questionable, and their opposition to organized labor is legendary. Some Syndicate factory towns have even been accused of human rights abuses.`
		log "Factions" "Pirates" `In poorer and more remote star systems, where the Navy seldom patrols, pirate attacks on merchant ships are frequent. Pirates are also known to attack large, unguarded convoys of freighters even in more populated areas. Most pirate fleets come from lawless worlds on the outskirts of human space.`

> 		

Reviewer 5: Since you won't get the First Contact mission, you won't get a log for the Hai.
```suggestion
		log "Factions" "Hai" `The Hai are a species of giant, intelligent rodents, who live to the north of human space. They allow any humans who discover their territory to live alongside them, and to trade with them and purchase their technology.`

```

event "not from boston"
	planet "New Boston"

> 		description `Most of the land here is wet and marshy, and the coastal regions are prone to flooding. Some food crops are grown here, but most of the farmers instead plant flax and jute, which grow well in wet soil and are used to make linen and burlap cloth. The textile mills are hot and poorly ventilated, but they offer higher and more reliable income than farming.`

Reviewer 5: Should we be a little cheeky?
```suggestion
		description `New Boston appears to be a completely unremarkable world... but as you fly through its atmosphere, you get a sense of deja vu, as if you had once been familiar with this planet before.`
		description `	Most of the land here is wet and marshy, and the coastal regions are prone to flooding. Some food crops are grown here, but most of the farmers instead plant flax and jute, which grow well in wet soil and are used to make linen and burlap cloth. The textile mills are hot and poorly ventilated, but they offer higher and more reliable income than farming.`
```

event "grew up on greenwater"
	planet "Greenwater"

> 		add description `It feels good to be home.`

Reviewer 6: This sort of falls into the 'telling the player what they feel' issue. I would make this more similar to the New Boston start's description of New Boston.

Author: ```suggestion
		add description `This is also the place where you grew up.`
```

			term 730
	set "species: human"
	set "hai space start"
	# need to set it here to not have it trigger alongside the invisible mission
	set "First Contact: Hai: offered"
	set "Intro [0]: offered"

conversation "hai intro"
	scene "scene/hai shipyard"
	`You've never been to the shipyard before, and going to the North Pole already feels like you're traveling to another world. The pictures and videos you've seen of all these ships before simply don't do them justice; seeing them for real is something else entirely. Even though you know a lot of those ships must have been in service for many years, you are stunned by how new and shiny they look. You begin to wonder if you are ready to take the big leap.`
	`	This "Aphid" could suit your needs. It is fast and able to carry a respectable amount of cargo.`
	scene "thumbnail/hai aphid"
	`	That is, unless you want to get adventurous and try out the impetuous Grasshopper. Surely the interceptor would be able to beat the smaller human warships, provided a competent pilot is at the helm.`
	`	You have, after all, heard tales of pirates and quick money from the many newcomers to Hai space - for those willing to take big risks, that is.`

> 	`	Alternatively, up north you can find the 'Unfettered'. Maybe they would come to like you, if you were to show them the strength and cunning they respect.`

Reviewer 5: ```suggestion
	`	Alternatively, up north you can go and meet the "Unfettered Hai." Maybe they would come to like you, if you were to show them the strength and cunning they respect.`
```

		log "Factions" "Hai" `The Hai are a species of giant, intelligent rodents, who live to the north of human space. They allow any humans who discover their territory to live alongside them, and to trade with them and purchase their technology.`

event "not from boston"
	planet "New Boston"
		description `New Boston appears to be a completely unremarkable world... but as you fly through its atmosphere, you get a sense of deja vu, as if you had once been familiar with this planet before.`
		description `	Most of the land here is wet and marshy, and the coastal regions are prone to flooding. Some food crops are grown here, but most of the farmers instead plant flax and jute, which grow well in wet soil and are used to make linen and burlap cloth. The textile mills are hot and poorly ventilated, but they offer higher and more reliable income than farming.`

event "grew up on greenwater"
	planet "Greenwater"

> 		add description `This is also the place where you grew up.`

Reviewer 6: I would beef this up a little. As of now this feels kind of tacked on. The description of New Boston in the vanilla start has a much more significant mention of it being your homeworld.

Author: I mean sure thing is I dont really know what to say

Reviewer 5: Maybe have a little bit about the planet from a local perspective like in the style of New Boston; culture, weather... (Probably I could suggest further changes to both New Boston and Greenwater descriptions)

Author: the thing is the description of greenwater already has these things in it and it's pretty fleshed out

Reviewer 5: Hmm...
> Greenwater is one of the few worlds in Hai space that is home to a sizable human population. The temperature on most of the planet's surface is too warm for the Hai, but tolerable for human beings. Close to the equator are scattered human settlements for farming or fishing, which trade with the Hai cities closer to the poles for technology and equipment.

If I gave it a shot:
> Greenwater is unique among Hai worlds, not only in that it is home to a sizable human population, but also because it happens to be the world where you grew up. Most of the humans here live on the islands and archipelagoes across the equator of Greenrock where the Hai seldom visit on account of their fur coats, but from what you hear from humans who have come from "beyond the wormhole," the climate of Greenwater is like a "Paradise Planet" even without terraforming.
> While farming and fishing make up most of the human-centered economy here, a few merchants trade with the Hai up in the poles where the spaceport is located.

Of course it could probably use with some improvements.

Author: I love it!
~~I'm prob not the best choice for improving it, so I'll just apply it~~

	date 16 11 3013
	to display
		or
			has "main plot completed"
			has "Ask the Hai about the Unfettered"
	on display
		description "What if you had not grown up on New Boston? What if you had waited longer before taking to the skies?"
		thumbnail ""
		date "3020"
		planet "Slightly too warm"
		credits "over a million"
	to reveal
		has "main plot completed"
		has "Ask the Hai about the Unfettered"
	on reveal
		name "Undiscovered Hai Start"
		description "There could be interesting possibilities up north, once all crises are resolved."
		description "	You could have started a different way, went a different path."
		system "Somewhere far north"
		planet "Mixed culture"
		date "Same old one"
		credits 1.431M
		debt 1.431M
	to unlock
		has "main plot completed"
		has "event: hai-human resolution announced"
	conversation "hai intro"
	account
		credits 1431000
		score 400
		mortgage Mortgage
			principal 1431000
			interest 0.003
			term 730
	set "species: human"
	set "hai space start"
	# need to set it here to not have it trigger alongside the invisible mission
	set "First Contact: Hai: offered"
	set "Intro [0]: offered"

conversation "hai intro"
	scene "scene/hai shipyard"

> 	`You've never been to the shipyard before, and going to the North Pole already feels like you're traveling to another world. The pictures and videos you've seen of all these ships before simply don't do them justice; seeing them for real is something else entirely. Even though you know a lot of those ships must have been in service for many years, you are stunned by how new and shiny they look. You begin to wonder if you are ready to take the big leap.`

Reviewer 5: Since you're from the equator, you'll probably not have seen so many Hai in one place. The player should probably comment on that, unless they've been to the North Pole before.

Also, where was this travel agency, when did you work for it? Did you need to work there to save up for a license? Was it up in the North Pole where the shipyard/spaceport is? It doesn't make too much sense if you "feel you're traveling to another world" if you work at a travel agency in the North Pole. If you don't mention it here, it'll just be confusing later on in the Project Hawking and Human to Hai Space missions.
```suggestion
	`You've never been to a shipyard before, and going to the North Pole already makes you feel as if you're on another world. Though you've met some Hai before, you've never seen so many of them in one place; the pace of this Hai city far outstrips the humid human town on the equator where you grew up.`
	`	The pictures and videos you've seen of all these ships before simply don't do them justice; seeing them for real is something else entirely. Even though you know a lot of those ships must have been in service for many years, you are stunned by how new and shiny they look. You begin to wonder if you are ready to take the big leap.`
```

Author: the agency wasnt on the pole, it was in the same mostly human city as that other dude you bring back to hai space in a mission

Reviewer 5: What do you think of the suggestion above, though?

Reviewer 5: It seems as if they haven't been to a Hai city, but otherwise, sure. I can change it.

Reviewer 5: ```suggestion
	`You've never been to the shipyard before, and going to the North Pole already makes you feel as if you're on another world. Though you've met some Hai in the past, you've never been among so many of them in one place; the pace of this Hai city far outstrips the muggy human town on the equator where you grew up.`
	`	The pictures and videos you've seen of all these ships before simply don't do them justice; seeing them for real is something else entirely. Even though you know a lot of those ships must have been in service for many years, you are stunned by how new and shiny they look. You begin to wonder if you are ready to take the big leap.`
```

	`	This "Aphid" could suit your needs. It is fast and able to carry a respectable amount of cargo.`
	scene "thumbnail/hai aphid"
	`	That is, unless you want to get adventurous and try out the impetuous Grasshopper. Surely the interceptor would be able to beat the smaller human warships, provided a competent pilot is at the helm.`
	`	You have, after all, heard tales of pirates and quick money from the many newcomers to Hai space - for those willing to take big risks, that is.`
	`	Alternatively, up north you can go and meet the "Unfettered Hai." Maybe they would come to like you, if you were to show them the strength and cunning they respect.`
	choice
		`	(Go to the bank.)`
	scene "scene/hai bank"
	`	You head to the bank's lobby, hoping to get enough of a loan to be able to buy one of these pristine ships.`
	`	To your disappointment, the only ship you could afford is an Aphid. You arrange the contract to loan as much money as you need in order to buy one, in what has to be the first seriously unwise act of your life.`

> 	`	You've heard of new captains stripping down their ships to recoup the initial buy-in, which is crossing your mind as you see the financial burden you're about to take on. As you wait for the paperwork to be processed, you passively scan over an Aphid pamphlet in the office. The pricey shield regenerator could be traded in if dangerous areas are avoided... and perhaps even the anti-missile system could be downgraded or outright removed... one of the Sand Cells could be sold back to make more space for cargo...`

Author: well it was about uhai space but yeah the player should know about them anyway

> 	`	Those anxious musings are interrupted when the bank employee finally lays it out to you, with a hesitant smile. "You are accepting a loan in the amount of 1,431,000 credits, due over the course of 2 years, with an interest rate of 0.3% per day. Please sign here."`

Author: I dont know honestly, could be either

	name

> 	`	You head out of the bank, but on your way to buy the Aphid at the shipyard, you see a notice. A retiring captain is offering to sell some of his ships at an interesting price, including a Grasshopper... Maybe they could sell it to you.`

Reviewer 5: "Interesting" price? I suppose it makes sense, but it could be better.

Is the player supposed to be so passive? Seems like they should be active if they're taking on such a big decision.
```suggestion
	`	You head out of the bank, but on your way to buy the Aphid at the shipyard, you see a notice. A retiring captain is selling off his fleet at a reduced price, and one of the ships he's offering is a Grasshopper... Maybe they could sell it to you.`
```
Or:
```suggestion
	`	You head out of the bank, but on your way to buy the Aphid at the shipyard, you see a notice. A retiring captain is selling off his fleet at a reduced price, and one of the ships he's offering is a Grasshopper... Maybe you could buy it.`
```

Reviewer 5: By the way, this is not really necessary, but it would be interesting (haha) to split the mission here and say "time to head to the shipyard to buy an aphid". Then use a shipyard mission to give the player the option of buying the Pristine Jewel.

	choice
		`	(Go to the shipyard.)`
			decline
		`	(Go see what the captain has to offer first.)`

> 	`	You head to the spaceport and easily find the captain selling off his ships. Thankfully he has not yet sold the Grasshopper.`

Reviewer 5: Again, since you're from the equator where the Hai don't usually frequent, you'll probably differentiate this captain as being a Hai. ~~It was also a little confusing.~~
```suggestion
	`	You head to the spaceport and easily find the captain, a Hai, selling one of his Centipedes to another Hai. Thankfully it looks like he has not yet sold the Grasshopper.`
```
I was also confused what the captain was doing. Was he idling around and waiting for buyers, or did he already have some? I like the second option.

Author: you want to add other ppl betting against you haha?

> 	`	The captain greets you in Hai. You are embarrassed to have to explain to them that you never learned their language. Most Hai speak your language however, and, as usual, they are able to switch to speaking yours without much difficulty.`

Reviewer 5: ```suggestion
	`	When he's finished selling the Centipede the captain turns and greets you in Hai. You are embarrassed to have to explain to them that you never learned their language. Most Hai speak your language however, and, as usual, they are able to switch to speaking yours without much difficulty.`
```

	`	"Hello, good fellow! I assume you come for the Grasshopper." Your interest for this ship must be apparent.`
	choice
		`	"Yes indeed."`
		`	"How did you know?"`
		`	"Sorry, not interested."`
			decline

> 	`	"I can see the will to explore in your eyes, but it is not only that: I know that some humans want to head back to their space, now that it is safer to do so. I would be delighted to have you flying the 'Pristine Jewel'!" they say, taking you to the ship. It is clearly not brand new, but is still in a surprisingly good condition.`

Reviewer 5: ```suggestion
	`	"I can see the will to explore in your eyes, but it is not only that: I know that some humans want to head back to their space, now that it is safer to do so. I would be delighted to have you flying the 'Pristine Jewel'!" he says, taking you to the ship. It is clearly not brand new, but is still in a surprisingly good condition.`
```
The Hai is referred to as "he" earlier on, might as well repeat it.

	scene "thumbnail/hai grasshopper"
	`	"Because I like you I'll give you a friendly price. What do you say about... a bit over a million credits, for this wonderful ship? That's less than a third the price you would have to pay if you were to buy this model from the shipyard. Except you can't find a model like this in shipyards, trust me!"`
	choice
		`	"Sold!"`
		`	"Sorry, not interested."`
			decline
	`	As the two of you settle up the paperwork, the captain takes the time to explain, in great detail, why this ship is so marvelous. In your excitement to buy the ship all you can hear from the Hai captain's speech is the repeated reference to the Grasshopper's 'exceptional' reinforced armor.`

> 	`	Before you leave, they give you a final tip: "If you want to, you can always sell this ship and buy an Aphid instead. The shipyard authorities will give you a discount for having sold my ship."`

Reviewer 5: ```suggestion
	`	Before you leave, he gives you a final tip: "If you want to, you can always sell this ship and buy an Aphid instead. The shipyard authorities will give you a discount for having sold my ship."`
```

			`The Syndicate is a central part of the Republic's economy, but they are also well known for selling shoddy products and for doing damage to the environment on the worlds they control. Their treatment of workers is questionable, and their opposition to organized labor is legendary. Some Syndicate factory towns have even been accused of human rights abuses.`
		log "Factions" "Pirates" `In poorer and more remote star systems, where the Navy seldom patrols, pirate attacks on merchant ships are frequent. Pirates are also known to attack large, unguarded convoys of freighters even in more populated areas. Most pirate fleets come from lawless worlds on the outskirts of human space.`
		log "Factions" "Hai" `The Hai are a species of giant, intelligent rodents, who live to the north of human space. They allow any humans who discover their territory to live alongside them, and to trade with them and purchase their technology.`

event "not from boston"
	planet "New Boston"

> 		description `New Boston appears to be a completely unremarkable world... but as you fly through its atmosphere, you get a sense of deja vu, as if you had once been familiar with this planet before.`
> 		description `	Most of the land here is wet and marshy, and the coastal regions are prone to flooding. Some food crops are grown here, but most of the farmers instead plant flax and jute, which grow well in wet soil and are used to make linen and burlap cloth. The textile mills are hot and poorly ventilated, but they offer higher and more reliable income than farming.`

Reviewer 5: I'll improve what I wrote, and try to change the parts that seem like it comes from a local perspective.
```suggestion
		description `New Boston appears to be a completely unremarkable world... but as you fly through its atmosphere, you get a sense of deja vu, as if you once knew this planet intimately.`
		description `	Most of the land here is wet and marshy, and the coastal regions are prone to flooding. The planet's economy is largely agricultural; some of the farms here grow food for export to Earth and elsewhere, but a significant portion of farming land is devoted to growing fiber crops, which is then processed in the textile mills in the spaceport city.`
```

> 

Reviewer 5: ```suggestion
	spaceport `New Boston's shabby spaceport is as about as basic as a spaceport can be. Many of the concrete landing pads are tilted and uneven, having sunk into the muddy soil since they were poured, and you have to be careful landing on them.`
	spaceport `	The market where ship captains buy and sell cargo is a large tent rather than a permanent building, but it is sturdy enough to keep off the rain. Loads are brought in and out by trucks and the occasional hovercraft. Here some enterprising local mechanics have set up not only a repair shop and outfitter, but also a shipyard with several small, refurbished ships for sale.`

```

event "grew up on greenwater"
	planet "Greenwater"

> 		description `Greenwater is unique among Hai worlds, not only in that it is home to a sizable human population, but also because it happens to be the world where you grew up. Most of the humans here live on the islands and archipelagoes across the equator of Greenrock where the Hai seldom visit on account of their fur coats, but from what you hear from humans who have come from "beyond the wormhole," the climate of Greenwater is like a "Paradise Planet", even without terraforming.`

Reviewer 6: ```suggestion
		description `Greenwater is unique among Hai worlds, not only in that it is home to a sizable human population, but also because it happens to be the world where you grew up. Most of the humans here live on the islands and archipelagoes across the equator of Greenrock where the Hai seldom visit on account of their fur coats. From what you hear from humans who have come from beyond the wormhole, the climate of Greenwater is like a "Paradise Planet", even though it lacks the extensive terraforming common to those worlds..`
```

> 		description `Greenwater is unique among Hai worlds, not only in that it is home to a sizable human population, but also because it happens to be the world where you grew up. Most of the humans here live on the islands and archipelagoes across the equator of Greenrock where the Hai seldom visit on account of their fur coats. From what you hear from humans who have come from beyond the wormhole, the climate of Greenwater is like a "Paradise Planet", even though it lacks the extensive terraforming common to those worlds..`
> 		description `	While farming and fishing make up most of the human-centered economy here, a few merchants trade with the Hai up in the poles where the spaceport is located.`

Reviewer 5: ~~Greenrock~~ oops.
```suggestion
		description `Greenwater is unique among Hai worlds, not only in that it is home to a sizable human population, but also because it happens to be the world where you grew up.`
		description `	Most of the humans here live on the islands and archipelagoes across the equator of Greenwater, where the Hai seldom visit on account of their heavy fur coats. From what you hear from humans who have come from beyond the wormhole, the climate here is like a "Paradise Planet," even though it lacks the extensive terraforming common to those worlds.`
		description `	While farming and fishing make up most of the human-centered economy, a few merchants trade with the Hai up in the poles where the spaceport is located.`
```

substitutions
	"<origin planet>" "New Boston"
	"<origin planet>" "Greenwater"

> 		has "hai space start"

Reviewer 7: Just to be clear, does having these in order cause the Hai one to supplant the default if applicable, or should the New Boston one have a `has "human space start"` condition on it as well?

Author: It'll be New Boston by default, and if the other condition is true then it'll be overwritten to Greenwater

	`	The only moment when he becomes truly animated is when explaining the Truth in Lending statement to you. "You are borrowing 480,000 credits," he says, "to be repaid over the course of one year. Your daily interest rate is 0.4%, which means that your daily payments are 2,503 credits, and by the end of the year you will have paid... 433,567 credits in interest. Sign here." He grins.`
	`	You sign your name one final time:`
	name
	`	The elevator is so well-tuned that you do not even realize it is moving until it has deposited you back in the lobby. But as you leave the bank, you are smiling. This crazy adventure suddenly feels real to you. You are going to do it. You are finally going to get off this planet.`
	`	Compared to the bank, you feel much more at home in the shipyard, walking among the rusted out hulks and newer ships that gleam in the sunlight. You smell grease and dirt and rocket fuel; wonderful smells. There are three ship models within your price range. Which one you choose will determine your future...`

start "hai space"

> 	name "New Hai"

Reviewer 5: ```suggestion
	name "Hai Space Start"
```
Seems more appropriate?

Author: The goal was to be original but yeah ig so haha

	description `You grew up on Greenwater, a world with a very rich and mixed culture. You've dreamed of owning your own fleet since you were given toy spaceships as a small child. In order to reach that goal, you worked hard at a travel company, recommending mostly locations in human space.`
	description `	Now, as you enter the shipyard, you feel like this is the start of a new life as a captain.`
	thumbnail "scene/hai start"
	system "Fah Soom"
	planet "Greenwater"
	date 16 11 3013
	to display
		or
			has "main plot completed"
			has "Ask the Hai about the Unfettered"
	on display

> 		description "What if you had not grown up on New Boston? What if you had waited longer before taking to the skies?"

Author: ```suggestion
		description "What if you had not grown up on New Boston?"
```

		thumbnail ""
		date "3020"
		planet "Slightly too warm"
		credits "over a million"
	to reveal
		has "main plot completed"
		has "Ask the Hai about the Unfettered"
	on reveal
		name "Undiscovered Hai Start"
		description "There could be interesting possibilities up north, once all crises are resolved."
		description "	You could have started a different way, went a different path."
		system "Somewhere far north"
		planet "Mixed culture"

> 		date "Same old one"

Author: ```suggestion
		date "16 11 3013"
```

		credits 1.431M
		debt 1.431M
	to unlock
		has "main plot completed"

> 		has "event: hai-human resolution announced"

Reviewer 5: This won't be unlockable until HR is finished. Do we want to change this, or leave it until that campaign does get finished?

Author: ```suggestion
		#has "event: hai-human resolution announced"
```
I'd just comment it out for now

Reviewer 8: Could you also add a TODO to it, so we can come back to it easier after HR?

> 		# has "event: hai-human resolution announced"

Author: ```suggestion
		# TODO: think about what point of HR should unlock this once it's back in the game
		# has "event: hai-human resolution announced"
```

	conversation "hai intro"
	account
		credits 1431000
		score 400
		mortgage Mortgage
			principal 1431000
			interest 0.003
			term 730
	set "species: human"
	set "hai space start"
	# need to set it here to not have it trigger alongside the invisible mission
	set "First Contact: Hai: offered"
	set "Intro [0]: offered"

conversation "hai intro"
	scene "scene/hai shipyard"

> 	`You've never been to the shipyard before, and going to the North Pole already makes you feel as if you're on another world. Though you've met some Hai in the past, you've never been among so many of them in one place; the pace of this Hai city far outstrips the muggy human town on the equator where you grew up.`

Reviewer 8: ```suggestion
	`You've never been to the shipyard before, and going to the North Pole already makes you feel as if you're on another world. Though you've met some Hai in the past, you've never been among so many of them in one place; the pace of this Hai city far outstrips the muggy human town near the equator where you grew up.`
```

> 	`You've never been to the shipyard before, and going to the North Pole already makes you feel as if you're on another world. Though you've met some Hai in the past, you've never been among so many of them in one place; the pace of this Hai city far outstrips the muggy human town near the equator where you grew up.`

Author: it is on Earth, I think?

	`	The pictures and videos you've seen of all these ships before simply don't do them justice; seeing them for real is something else entirely. Even though you know a lot of those ships must have been in service for many years, you are stunned by how new and shiny they look. You begin to wonder if you are ready to take the big leap.`
	`	This "Aphid" could suit your needs. It is fast and able to carry a respectable amount of cargo.`
	scene "thumbnail/hai aphid"
	`	That is, unless you want to get adventurous and try out the impetuous "Grasshopper." Surely the Hai interceptor could take on any of the small, human warships here... provided a competent pilot is at the helm.`

> 	`	You have, after all, heard tales of pirates and quick money from the many newcomers to Hai space - for those willing to take big risks, that is.`

Reviewer 6: ```suggestion
	`	You have heard tales of pirates and quick money from the many newcomers to Hai space - for those willing to take big risks, that is.`
```

	`	Alternatively, up north you can go and meet the "Unfettered Hai." Maybe they would come to like you, if you were to show them the strength and cunning they respect.`
	choice
		`	(Go to the bank.)`
	scene "scene/hai bank"
	`	You head to the bank's lobby, hoping to get enough of a loan to be able to buy one of these pristine ships.`

> 	`	To your disappointment, the only ship you could afford is an Aphid. You arrange the contract to loan as much money as you need in order to buy one, in what has to be the first seriously unwise act of your life.`

Reviewer 5: ```suggestion
	`	To your disappointment, the only ship you can afford is an Aphid. You arrange the contract to loan as much money as you need in order to buy one, in what has to be the first seriously unwise act of your life.`
```
Past tense jumpscare.

	`	You've heard of new captains stripping down their ships to recoup the initial buy-in, which is crossing your mind as you see the financial burden you're about to take on. As you wait for the paperwork to be processed, you passively scan over an Aphid pamphlet in the office. The pricey shield regenerator could be traded in if you can avoid combat... and perhaps even the anti-missile system could be downgraded or outright removed... one of the Sand Cells could be sold back to make more space for cargo...`
	`	Those anxious musings are interrupted when the bank employee finally lays it out to you, with a hesitant smile. "You are accepting a loan in the amount of 1,431,000 credits, due over the course of 2 years, with an interest rate of 0.3% per day. Please sign here."`
	name

> 	`	You head out of the bank, but on your way to buy the Aphid at the shipyard, you see a notice. A retiring captain is selling off his fleet at a reduced price, and one of the ships he's offering is a Grasshopper... Maybe you could buy it.`

Reviewer 5: ```suggestion
	`	You head out of the bank, but on your way to buy the Aphid at the shipyard, you see a notice. A retiring captain is selling off his fleet at a reduced price, and one of the ships he's offering is a Grasshopper... maybe you could buy it.`
```
idk if I changed the capitalisation in the past but I'm doing it again

> 	`	You head out of the bank, but on your way to buy the Aphid at the shipyard, you see a notice. A retiring captain is selling off his fleet at a reduced price, and one of the ships he's offering is a Grasshopper... maybe you could buy it.`

Reviewer 8: ```suggestion
	`	You head out of the bank, but on your way to buy the Aphid at the shipyard, you see a notice. A retiring captain is selling off his fleet at a reduced price, and one of the ships he's offering is a Grasshopper... maybe you could get a deal on it.`
```

	choice
		`	(Go to the shipyard.)`
			decline
		`	(Go see what the captain has to offer first.)`

> 	`	You head to the spaceport and easily find the captain, a Hai, selling one of his Centipedes to another Hai. Thankfully it looks like he has not yet sold the Grasshopper.`

Reviewer 8: ```suggestion
	`	You head to the spaceport and easily find the captain, a Hai, selling one of his Centipedes to another Hai. Thankfully it looks like he has not sold the Grasshopper yet.`
```

	`	When he's finished selling the Centipede the captain turns and greets you in Hai. You are embarrassed to have to explain to them that you never learned their language. Most Hai speak your language however, and, as usual, they are able to switch to speaking yours without much difficulty.`
	`	"Hello, good fellow! I assume you come for the Grasshopper." Your interest for this ship must be apparent.`
	choice

> 		`	"Yes indeed."`

Reviewer 8: ```suggestion
		`	"Indeed."`
```

	description `You grew up on Greenwater, a world with a very rich and mixed culture. You've dreamed of owning your own fleet since you were given toy spaceships as a small child. In order to reach that goal, you worked hard at a travel company, recommending mostly locations in human space.`
	description `	Now, as you enter the shipyard, you feel like this is the start of a new life as a captain.`
	thumbnail "scene/hai start"
	system "Fah Soom"
	planet "Greenwater"
	date 16 11 3013
	to display
		or
			has "main plot completed"
			has "Ask the Hai about the Unfettered"
	on display
		description "What if you had not grown up on New Boston? What if you had waited longer before taking to the skies?"
		thumbnail ""
		date "3020"
		planet "Slightly too warm"
		credits "over a million"
	to reveal
		has "main plot completed"
		has "Ask the Hai about the Unfettered"
	on reveal
		name "Undiscovered Hai Start"
		description "There could be interesting possibilities up north, once all crises are resolved."
		description "	You could have started a different way, went a different path."
		system "Somewhere far north"
		planet "Mixed culture"
		date "Same old one"
		credits 1.431M

> 		debt 1.431M

Author: Idk about this
Yes I probably added more than what was needed, but I feel like removing all of it is too much

	choice
		`	"Indeed."`
		`	"How did you know?"`
		`	"Sorry, not interested."`
			decline
	`	"I can see the will to explore in your eyes, but it is not only that: I know that some humans want to head back to their space, now that it is safer to do so. I would be delighted to have you flying the 'Pristine Jewel'!" he says, taking you to the ship. It is clearly not brand new, but is still in a surprisingly good condition.`
	scene "thumbnail/hai grasshopper"
	`	"Because I like you I'll give you a friendly price. What do you say about... a bit over a million credits, for this wonderful ship? That's less than a third the price you would have to pay if you were to buy this model from the shipyard. Except you can't find a model like this in shipyards, trust me!"`
	choice
		`	"Sold!"`
		`	"Sorry, not interested."`
			decline
	`	As the two of you settle up the paperwork, the captain takes the time to explain, in great detail, why this ship is so marvelous. In your excitement to buy the ship all you can hear from the Hai captain's speech is the repeated reference to the Grasshopper's 'exceptional' reinforced armor.`

> 	`	Before you leave, he gives you a final tip: "If you want to, you can always sell this ship and buy an Aphid instead. The shipyard authorities will give you a discount for having sold my ship."`

Reviewer 4: Talking about how the player doesn't know Hai or talking about how the Grasshopper is customised are some of the most boring things you could do in this situation.

The player not knowing Hai is such a generic detail that you could write almost any interaction with a Hai and put that line in. About the only place where it'd be more than just a band-aid explanation over a plot hole is maybe in the bank scene, where you could get some tension about not knowing the Hai language when dealing with very important legal papers.

The Grasshopper modifications are more relevant to the situation, but the line's only purpose is to go "HEY! THE STATS ON THIS VIDEO GAME SHIP ARE DIFFERENT FROM THE STATS OF OTHER SIMILAR SHIPS IN THIS VIDEO GAME", which isn't great for immersion. Obviously, you do need to convey this info to the player, but you can just sneak it in as an aside instead of making it the ship's main description.

What hasn't been explored at all in this segment is why he's even retiring in the place. Is it because of old age? Has the lifestyle become boring to him? Was he traumatised by a freak Unfettered attack? Is the rationale behind his retirement the same as a human might have?

And also, what are people doing in response to the captain selling of his fleet? (Besides the obvious answer of "buying his ships".) Do they see it as a noble way to give back to the people? Or do they think that it's a gaudy stunt, and that he should just be selling it straight to a shipyard? Are they picking up his ships left, right and centre, or is he having to offer ridiculous discounts just to get rid of his ships? How does Hai society interact with the situation in a way that human society doesn't?

Answering these questions is what will make this start a proper introduction to the universe from the perspective of a human in Hai space, and not just a linear sequence of events and thoughts that are inflicted upon the player.

Author: I do need to say the player doesn't speak hai, or give them a translator which potentially wouldn't mix that well with campaigns and all, as only the uhai one takes that into account that I know of

	action
		set "hai space start: Grasshopper"
		give ship "Grasshopper (Expensive)" "Pristine Jewel"
		payment -1132000

# this ship uses the most expensive loadout possible (3.5M)
# so that you +- have the same amount of value back if you sell it (1.3mill)
# It has a small hull buff to help the player survive.
ship "Grasshopper" "Grasshopper (Expensive)"
	add attributes
		"hull" 400
	outfits
		`"Benga" Atomic Thruster`
		`"Biroo" Atomic Steering`
		"Fuel Pod"
		"Hai Chasm Batteries"
		"Hai Corundum Regenerator"
		"Hai Williwaw Cooling"
		Hyperdrive
		"Ion Cannon"
		"Pebble Core"
		"Pulse Rifle"
		"Quantum Keystone" 2

# the start itself cannot initiate events
mission "Greenwater start"
	landing
	invisible
	to offer
		has "hai space start"
	on offer

> 		event "not from boston"
> 		event "grew up on greenwater"

Reviewer 5: ```suggestion
		event "not from boston" 0
		event "grew up on greenwater" 0
```
Once you sync with master these will update after you leave the shipyard.

			`The Syndicate is a central part of the Republic's economy, but they are also well known for selling shoddy products and for doing damage to the environment on the worlds they control. Their treatment of workers is questionable, and their opposition to organized labor is legendary. Some Syndicate factory towns have even been accused of human rights abuses.`
		log "Factions" "Pirates" `In poorer and more remote star systems, where the Navy seldom patrols, pirate attacks on merchant ships are frequent. Pirates are also known to attack large, unguarded convoys of freighters even in more populated areas. Most pirate fleets come from lawless worlds on the outskirts of human space.`
		log "Factions" "Hai" `The Hai are a species of giant, intelligent rodents, who live to the north of human space. They allow any humans who discover their territory to live alongside them, and to trade with them and purchase their technology.`

event "not from boston"
	planet "New Boston"

> 		description `New Boston appears to be a completely unremarkable world... but as you fly through its atmosphere, you get a sense of deja vu, as if you once knew this planet intimately.`
> 		description `	Most of the land here is wet and marshy, and the coastal regions are prone to flooding. The planet's economy is largely agricultural; some of the farms here grow food for export to Earth and elsewhere, but a significant portion of farming land is devoted to growing fiber crops, which is then processed in the textile mills in the spaceport city.`

Reviewer 4: Subtlety. Also, this wouldn't make sense if we ever get alternate human starts that are available at the beginning.
```suggestion
		description `New Boston is a completely unremarkable world. Most of the land here is wet and marshy, and the coastal regions are prone to flooding. The planet's economy is largely agricultural; some of the farms here grow food for export to Earth and elsewhere, but a significant portion of farming land is devoted to growing fiber crops, which is then processed in the textile mills in the spaceport city.`
```

Reviewer 5: ~~R.I.P.~~

	`	The only moment when he becomes truly animated is when explaining the Truth in Lending statement to you. "You are borrowing 480,000 credits," he says, "to be repaid over the course of one year. Your daily interest rate is 0.4%, which means that your daily payments are 2,503 credits, and by the end of the year you will have paid... 434,000 credits in interest. Sign here." He grins.`
	`	You sign your name one final time:`
	name
	`	The elevator is so well-tuned that you do not even realize it is moving until it has deposited you back in the lobby. But as you leave the bank, you are smiling. This crazy adventure suddenly feels real to you. You are going to do it. You are finally going to get off this planet.`
	`	Compared to the bank, you feel much more at home in the shipyard, walking among the rusted out hulks and newer ships that gleam in the sunlight. You smell grease and dirt and rocket fuel; wonderful smells. There are three ship models within your price range. Which one you choose will determine your future...`

start "hai"
	name "Later Start"

> 	description `You grew up on Greenwater, a world with a very rich and diverse hai-human culture. You've dreamed of owning your own fleet since you were given toy spaceships as a small child. In order to reach that goal, you worked hard in a texture farm, making human clothing that quite surprisingly sold very well.`

Reviewer 9: ```suggestion
	description `You grew up on Greenwater, a world with a very rich and diverse hai-human culture. You've dreamed of owning your own fleet since you were given toy spaceships as a small child. In order to reach that goal, you worked hard in a texture farm making human clothing that, quite surprisingly, sold very well.`
```

	description `	A lot has happened whilst you were on the planet. Upon entering the shipyard you finally feel that there is nothing to hold you back anymore.`
	thumbnail "scene/hai start"
	date 18 6 3020
	to display
		or
			has "main plot completed"
			has "Ask the Hai about the Unfettered"
	hint "What if you had not grown up on New Boston? What if you had waited longer before taking to the skies?"
	to reveal
		has "main plot completed"
		has "Ask the Hai about the Unfettered"
	to unlock
		has "main plot completed"
		has "event: hai-human resolution announced"
	system "Fah Soom"
	planet "Greenwater"
	conversation "hai intro"
	account
		credits 1431000
		score 400
		mortgage Mortgage
			principal 1431000
			interest 0.003
			term 730
	set "hai start"
	# need to set it here to not have it trigger alongside the invisible mission
	set "First Contact: Hai: offered"
	set "Intro [0]: offered"
	set "skip FW"
	set "skip HR"

conversation "hai intro"
	scene "scene/hai shipyard"

> 	`	You've never been to the shipyard before, and it feels like you're already traveling to another world. You wonder if you are ready to take the big leap... You've seen pictures of ships before, but seeing them for real is something else entirely. You are stunned by how new and shiny they look, even though you know a lot of those ships must have been in service for many years.`

Reviewer 9: ```suggestion
	`	You've never been to the shipyard before, and as you arrive, it already feels like you're traveling to another world. The pictures and video you've seen of all these ships before simply don't do them justice; seeing them for real is something else entirely. Even though you know a lot of those ships must have been in service for many years, you are stunned by how new and shiny they look. You begin to wonder if you are ready to take the big leap.`
```

> 	`	This "Aphid" could suit your needs. It is Fast, and able to carry a respectable amount of cargo.`

Reviewer 9: ```suggestion
	`	This "Aphid" could suit your needs. It is fast and able to carry a respectable amount of cargo.`
```

	scene "thumbnail/hai aphid"

> 	`	That is, except if you want to get adventurous and try out the impetious Grasshopper. Surely the interceptor would be able to beat human small warships prodided a competent pilot is at the helm...`

Reviewer 9: ```suggestion
	`	That is, unless you want to get adventurous and try out the impetuous Grasshopper. Surely the interceptor would be able to beat the smaller human warships, provided a competent pilot is at the helm.`
```

> 	`	You have heard tales of pirates and quick money... Indeed, up north you can find the 'Unfettered'. Maybe they could come to like you, if you were to show them strength...`

Reviewer 9: ```suggestion
	`	You have, after all, heard tales of pirates and quick money... And up north you can find the 'Unfettered'. Maybe they could come to like you, if you were to show them the strength and cunning they respect.`
```

	choice
		`	(Go to the bank.)`
	scene "scene/hai bank"
	`	You head to the bank's lobby, hoping to get enough of a loan to be able to buy one of these pristine ships.`
	`	To your disapointment the only ship you could afford is an Aphid. You arrange the contract to loan as much money as you need in order to buy one, in what has to be the first seriously unwise act of your life. You may have to sell some part of the ship in order to be able to afford flying it!`
	`	You can probably sell the shield regenerator without any consequences, so long as you do not get too close to hostile territory. You may even downgrade the anti-missile turret.`
	`	The bank employee sums it up to you, with a hesitant smile. "This will put you in a debt of 1 million 431 thousand credits, due in the course of 2 years, with an interest rate of 0.3% per day. Please sign here."`
	name

> 	`	You head out of the bank, on your way to buy the Aphid, but you see a notice. A retiring captain offers to sell some of his ships at an interesting price, including a Grasshopper... Maybe they could sell it to you.`

Reviewer 9: ```suggestion
	`	You head out of the bank, but on your way to buy the Aphid at the shipyard, you see a notice. A retiring captain is offering to sell some of his ships at an interesting price, including a Grasshopper... Maybe they could sell it to you.`
```

	choice
		`	(Go to the shipyard.)`
			decline
		`	(Go see what the captain has to offer first.)`

> 	`	You head to the spaceport and easily find the captain that offers to sell the Grasshopper. Thankfully they did not yet sell it yet.`

Reviewer 9: ```suggestion
	`	You head to the spaceport and easily find the captain selling off his ships. Thankfully they did not yet sell the Grasshopper.`
```

> 	`	The captain greets you in Hai. You somehow never really learned the language, probably due to the fact everyone speaks yours to some degree. Seeing you do not understand they switch to your language.`

Reviewer 9: ```suggestion
	`	The captain greets you in Hai, but you've somehow managed to never really learn the language. Most everyone speaks yours to at least some degree anyway, and once they realize you do not understand, they switch with little effort.`
```

	`	"Hello, good fellow! I assume you come for the Grasshopper." Your interest for this ship must be apparent.`
	choice
		`	"Yes indeed."`
		`	"How did you know?"`
		`	"Sorry, not interested."`
			decline

> 	`	"I can see the will to explore in your eyes. I would be delighted to have you flying the 'Pristine Jewel'!" They say, taking you to the ship. It is not brand new, but it still looks in surprisingly good condition.`

Reviewer 9: ```suggestion
	`	"I can see the will to explore in your eyes. I would be delighted to have you flying the 'Pristine Jewel'!" they say, taking you to the ship. It is clearly not brand new but still looks in surprisingly good condition.`
```

	scene "thumbnail/hai grasshopper"

> 	`	"Because I like you I'll give you a friendly price. What do you say about... 1 millions, for this wonderful ship? That's less than a third price you would have to pay if you were to buy this model in a shipyard, except you can't find a model like this in shipyards, trust me!"`

Reviewer 9: ```suggestion
	`	"Because I like you I'll give you a friendly price. What do you say about... one million flat, for this wonderful ship? That's less than a third the price you would have to pay if you were to buy this model from the shipyard. Except you can't find a model like this in shipyards, trust me!"`
```

	choice
		`	"Sold!"`
		`	"Sorry, not interested."`
			decline

> 	`	They take the time to explain why this ship is so marvelous but you are not listening anymore, because of how excited you are to finally fly`

Reviewer 9: ```suggestion
	`	As the two of you settle up the paperwork, the captain takes the time to explain, in great detail, why this ship is so marvelous. Because of how excited you are to finally fly, you miss everything other than repeated reference to the Ion Cannon.`
```

Author: this is where we would add details about any upgrades the ship would have (like extra shields?)

	action
		set "hai start: Grasshopper"
		give ship "Grasshopper (Expensive)" "Pristine Jewel"
		payment -1000000

> ship "Grasshopper" "Grasshopper (Expensive)"

Reviewer 9: Why is this called "(Expensive)" when it is, in fact, on sale? lol "(Custom)" or "(Retrofit") or something akin would probably be better.

	add attributes

> 		"outfit space" 30

Reviewer 9: ```suggestion
		"outfit space" 30
```

We should at least be also taking cargo space here. I know it's not just a starter ship but also an "accelerated" start as vanilla experience, so I'm okay with it being a bit extra nice, but it should still trade instead of simply gain.

20 cargo space seems justified, as it's still twice as efficient as what we can purchase normally, plus the added weapon space as well.

Reviewer 9: Ahhh! I was thinking this was additions above and beyond the proposed buff. I thought you were including the base ship changes, you noted including them lol

Comment before this one makes more sense now too. :)

Author: added a comment to clear things out and 400 extra hull
you cant make anything with that since you cant really regen it but it should you survive a bit

	`	The only moment when he becomes truly animated is when explaining the Truth in Lending statement to you. "You are borrowing 480,000 credits," he says, "to be repaid over the course of one year. Your daily interest rate is 0.4%, which means that your daily payments are 2,503 credits, and by the end of the year you will have paid... 433,567 credits in interest. Sign here." He grins.`
	`	You sign your name one final time:`
	name
	`	The elevator is so well-tuned that you do not even realize it is moving until it has deposited you back in the lobby. But as you leave the bank, you are smiling. This crazy adventure suddenly feels real to you. You are going to do it. You are finally going to get off this planet.`
	`	Compared to the bank, you feel much more at home in the shipyard, walking among the rusted out hulks and newer ships that gleam in the sunlight. You smell grease and dirt and rocket fuel; wonderful smells. There are three ship models within your price range. Which one you choose will determine your future...`

start "hai space"
	name "Hai Origins"

> 	description `You grew up on Greenwater, a world with a rich and mixed culture. You've dreamed of owning your own fleet since you were given toy spaceships as a small child. In order to reach that goal, you worked hard at a travel company, recommending locations in human space.`

Reviewer 4: ```suggestion
	description `You grew up on Greenwater, a Hai world with a rich and mixed culture thanks to its climate. You've dreamed of owning your own fleet since you were given toy spaceships as a small child. To reach that goal, you worked hard at a travel company, recommending locations in human space.`
```

	description `	Now, as your ferry docks at the main spaceport, you feel like this is the start of a new life as a captain.`
	thumbnail "scene/hai start"
	system "Fah Soom"
	planet "Greenwater"
	date 16 11 3013
	to reveal
		has "Ask the Hai about the Unfettered: offered"
	on reveal
		name "Hai Origins"

> 		description `You grew up on Greenwater, in one of the human cities built on the equator, where the climate is too hot for the Hai. You've dreamed of owning your own fleet since you were given toy spaceships as a small child. In order to reach that goal, you worked hard at a travel company, recommending locations in human space.`

Reviewer 4: ```suggestion
	description `You grew up on Greenwater, a Hai world with a rich and mixed culture thanks to its climate. You've dreamed of owning your own fleet since you were given toy spaceships as a small child. To reach that goal, you worked hard at a travel company, recommending locations in human space.`
```

		description `	Now, as your ferry docks at the main spaceport, you feel like this is the start of a new life as a captain.`
		system "Fah Soom"
		planet "Greenwater"
		date 16 11 3013
		credits 1431000
		debt 1431000
	to unlock
		has "main plot completed"
		has "Ask the Hai about the Unfettered: offered"
		# TODO: think about what point of HR should unlock this once it's back in the game, if any
		# has "event: hai-human resolution announced"
	conversation "hai intro"
	account
		credits 1431000
		score 400
		mortgage Mortgage
			principal 1431000
			interest 0.003
			term 730
	set "species: human"
	set "hai space start"

> 	# need to set it here to not have it trigger alongside the invisible mission
> 	set "First Contact: Hai: offered"
> 	set "Intro [0]: offered"

Reviewer 4: `"Intro [0]"` should have its offer conditions changed to not offer if you're not on the default start. Doing it this way breaks `"Default Start Patch"`.
I think you also need the conditions for the other Hai first contact missions offering (which I thought was already done here? I might've accidentally changed it).
```suggestion
	# need to set it here to not have it trigger alongside the invisible mission
	set "First Contact: Hai: offered"
	set "First Contact: Unfettered: offered"
	set "Ask the Hai about the Unfettered: done"
```

Author: it wasn't the case for the unfettered first contact but I guess one could assume you know about them already as part of working in a travel agency

		log "Factions" "Syndicate" `The Syndicate is a megacorporation, the largest employer in human space. People who cannot find steady work elsewhere flock to the Syndicate factory worlds, where they are almost guaranteed to find a job. Although the Syndicate is technically part of the Republic, Syndicate worlds are governed directly by the corporation.`
			`The Syndicate is a central part of the Republic's economy, but they are also well known for selling shoddy products and for doing damage to the environment on the worlds they control. Their treatment of workers is questionable, and their opposition to organized labor is legendary. Some Syndicate factory towns have even been accused of human rights abuses.`
		log "Factions" "Pirates" `In poorer and more remote star systems, where the Navy seldom patrols, pirate attacks on merchant ships are frequent. Pirates are also known to attack large, unguarded convoys of freighters even in more populated areas. Most pirate fleets come from lawless worlds on the outskirts of human space.`

> 		log "Factions" "Hai" `The Hai are a species of giant, intelligent rodents, who live to the north of human space. They allow any humans who discover their territory to live alongside them, and to trade with them and purchase their technology.`

Reviewer 4: Missing logbook entries.
```suggestion
		log "Factions" "Hai" `The Hai are a species of giant, intelligent rodents, who live to the north of human space. They allow any humans who discover their territory to live alongside them, and to trade with them and purchase their technology.`
		log "Factions" "Unfettered Hai" `The "Unfettered" are a faction within the Hai species who are not as peaceful as the rest, and are normally not friendly toward humans. They claim that long ago the Drak altered the Hai to make them docile and peaceful, and that the Unfettered are descendants of those Hai who avoided undergoing that alteration. They are seeking jump drives so that they can escape the "prison" they are now stuck in: a few deteriorating worlds on the far northern edge of Hai space.`
		log "Factions" "Sheragi" `A long extinct dragon-like species called the "Sheragi" used to live in human space, back when much of what is now human territory was occupied by the Hai.`
```

	`	Making your way towards the shipyard, you pull up the human-language version of your loan agreement to review: You have borrowed 1.431 million credits to be repaid over the course of two years. Your interest rate is 0.3%, making your repayments 4,836 credits per day.`
	`	You look up from your tablet just before running head-on into a gray-furred Hai wearing a flight suit, knocking him over. As you bend over to help, he takes a glance at your mortgage documents. "You're about to become a captain, aren't you?" he says after dusting himself off.`
	
	choice
		`	"Yes, I am."`
		`	"How could you tell?"`
			goto tell
		`	"Who said you could look at that?"`
			goto look
	
	`	"Then maybe I can help. Follow me."`
		goto help
	
	label tell
	`	"From your loan," he replies. "There's only one reason to have that exact number of credits. Speaking of which, I think I can help you."`
		goto help
	
	label look
	`	He scratches the back of his head. "Sorry. I was just trying to help, but if I'm causing too much trouble, I'll get out of your way.`
	choice
		`	"I don't mind."`
			goto happy
		`	"I'd rather handle things myself."`
			goto leave
	label happy
	`	He bows. "Thank you."`
	
	label help

> 	`	The Hai brings you to the landing pads of Greenwater, where a large number of humans and Hai are gathered around several vessels. From the crowd, another Hai approaches your companion, talking to him in Hai at such a speed that you'd have trouble understanding even if you knew the language. However, the gray-haired Hai simply nods and hands the other Hai a set of keys in exchange for a credit chip. The other Hai quickly bows before dashing towards one of the landed ships.`

Reviewer 4: For some reason, I forgot that the Hai language stuff was already brought up here, before the wormhole mission. I think we only really need one place where we point that out.
```suggestion
	`	The Hai brings you to the landing pads of Greenwater, where a large number of humans and Hai are gathered around several vessels. From the crowd, another Hai approaches your companion, talking to him in Hai. The gray-furred Hai simply nods and hands the other Hai a set of keys in exchange for a credit chip. The other Hai quickly bows before dashing towards one of the landed ships.`
```

		payment -1132000
	`	The Hai gives you the keys to the Grasshopper after you hand over the tablet and your payment. "Looks like this your first step to becoming an interstellar entrepreneur. I'd say we'll meet again someday, but I'd rather not lie. In all my years, I never did meet the captain who sold me my first ship again. Not that should matter, anyway; this is your story, not mine."`
	`	He gives you one last salute before returning to the crowd to sell yet another ship.`
		decline
	
	label leave
	`	He nods as you head towards the shipyard.`
	`	At its entrance is an automated security kiosk. You quickly fill out the form on the screen before picking up the stylus and signing your name:`
	name
	`	You step through glass sliding doors into Greenwater's shipyard. In most of the berths, workers are busy refurbishing old vessels for resale, but in some of the more distant bays, you can see the flash of welding torches on brand-new ship skeletons. Vessels of all sizes are lined up within the yard, but for you, there's only one option...`

# This ship uses the most expensive loadout possible so its sell price is roughly equal to the buy price.

> ship "Grasshopper" "Grasshopper (Expensive)"

Reviewer 4: ```suggestion
ship "Grasshopper" "Grasshopper (Hai Start)"
```

		spaceport `New Boston's shabby spaceport is as about as basic as a spaceport can be. Many of the concrete landing pads are tilted and uneven, having sunk into the muddy soil since they were poured, and you have to be careful landing on them.`
		spaceport `	The market where ship captains buy and sell cargo is a large tent rather than a permanent building, but it is sturdy enough to keep off the rain. Loads are brought in and out by trucks and the occasional hovercraft. Here some enterprising local mechanics have set up not only a repair shop and outfitter, but also a shipyard with several small, refurbished ships for sale.`

event "grew up on greenwater"
	planet "Greenwater"
		description `Greenwater is unique among Hai worlds, not only in that it is home to a sizable human population, but also because it happens to be the world where you grew up.`

> 		description `	Most of the humans here live as fishers and farmers on islands and archipelagoes across the equator of Greenwater, where the Hai seldom visit on account of their heavy fur coats. According to those beyond the wormhole, the climate here is like a "Paradise Planet," even though it lacks the extensive terraforming common to those worlds.`

Reviewer 6: ```suggestion
		description `	Most of the humans here live as fishers and farmers on islands and archipelagoes across the equator of Greenwater, where the Hai seldom visit on account of their heavy fur coats. According to those beyond the wormhole, the climate here is like a Paradise Planet, even though it lacks the extensive terraforming common to those worlds.`
```

> 		description `	Thanks to its high human population, human lanaguage is dominant on Greenwater, with Hai language limited to the poles of the planet where Hai merchants are more frequent.`

Reviewer 4: ```suggestion
		description `	Thanks to its high human population, human language is dominant on Greenwater, with Hai language limited to the poles of the planet where Hai merchants are more frequent.`
```

> 		description `	Thanks to its high human population, human language is dominant on Greenwater, with Hai language limited to the poles of the planet where Hai merchants are more frequent.`

Reviewer 6: ```suggestion
		description `	Thanks to its high human population, the human language is dominant on Greenwater, with the Hai language limited to the poles of the planet where Hai merchants are more frequent.`
```

> # Copyright (c) 2015 by Michael Zahniser

Author: fair point, thanks!
```suggestion
# Copyright (c) 2015 by Michael Zahniser
# Copyright (c) 2024 by Hurleveur
# Copyright (c) 2024 by Anarchist2
```
@Anarchist2 do you want in?

		log "Factions" "Pirates" `In poorer and more remote star systems, where the Navy seldom patrols, pirate attacks on merchant ships are frequent. Pirates are also known to attack large, unguarded convoys of freighters even in more populated areas. Most pirate fleets come from lawless worlds on the outskirts of human space.`
		log "Factions" "Hai" `The Hai are a species of giant, intelligent rodents, who live to the north of human space. They allow any humans who discover their territory to live alongside them, and to trade with them and purchase their technology.`
		log "Factions" "Unfettered Hai" `The "Unfettered" are a faction within the Hai species who are not as peaceful as the rest, and are normally not friendly toward humans. They claim that long ago the Drak altered the Hai to make them docile and peaceful, and that the Unfettered are descendants of those Hai who avoided undergoing that alteration. They are seeking jump drives so that they can escape the "prison" they are now stuck in: a few deteriorating worlds on the far northern edge of Hai space.`

> 		log "Factions" "Sheragi" `A long extinct dragon-like species called the "Sheragi" used to live in human space, back when much of what is now human territory was occupied by the Hai.`

Author: idk about this https://www.collinsdictionary.com/us/dictionary/english/long-extinct

> 		log "Factions" "Sheragi" `A long extinct dragon-like species called the "Sheragi" used to live in human space, back when much of what is now human territory was occupied by the Hai.`

Reviewer 3: Alright, good point.
~~except the URL uses a dash~~

	`	You sign your name one final time:`
	name
	`	The elevator is so well-tuned that you do not even realize it is moving until it has deposited you back in the lobby. But as you leave the bank, you are smiling. This crazy adventure suddenly feels real to you. You are going to do it. You are finally going to get off this planet.`
	`	Compared to the bank, you feel much more at home in the shipyard, walking among the rusted out hulks and newer ships that gleam in the sunlight. You smell grease and dirt and rocket fuel; wonderful smells. There are three ship models within your price range. Which one you choose will determine your future...`

start "hai"
	name "Hai Later Start"
	description `You grew up on Greenwater, a world with a very rich and diverse hai-human culture. You've dreamed of owning your own fleet since you were given toy spaceships as a small child. In order to reach that goal, you worked hard in a texture farm making human clothing that, quite surprisingly, sold very well.`

> 	description `	A lot has happened whilst you were on the planet. Upon entering the shipyard you finally feel that there is nothing to hold you back anymore.`

Reviewer 8: The first sentence doesn't really work.

Author: The issue here is I'm limited by space but I want to say FW and HR happened already
```suggestion
	description `	You heard stories of heroes saving both human and Hai. Now, as you enter the shipyard, you finally feel free. It is your turn.`
```

Reviewer 8: Honestly, I still don't really think these two sentences fit in the same paragraph, but space could indeed be an issue.

Author: yeah I just cant get another paragraph going I'm at the word limit (at one or two word close xD)

Author: how is this now? 
![image](https://user-images.githubusercontent.com/94366726/214263240-28770083-03d2-4307-81b6-3e832a09e0a0.png)

	thumbnail "scene/hai start"
	system "Fah Soom"
	planet "Greenwater"
	date 18 6 3020
	to display
		or
			has "main plot completed"
			has "Ask the Hai about the Unfettered"
	on display
		description "What if you had not grown up on New Boston? What if you had waited longer before taking to the skies?"
		thumbnail ""
		date "middle of 3020"
	to reveal
		has "main plot completed"
		has "Ask the Hai about the Unfettered"
	on reveal
		name "Hai Start"

> 		description "There could be interesting possibilities up north, once all crisies are resolved."

Reviewer 8: ```suggestion
		description "There could be interesting possibilities up north, once all crises are resolved."
```

		description "You could have started a different way, went a different path."
		date "Jun 3020"
		credits 1.431M
		debt 1.431M
	to unlock
		has "main plot completed"
		has "event: hai-human resolution announced"
	conversation "hai intro"
	account
		credits 1431000
		score 400
		mortgage Mortgage
			principal 1431000
			interest 0.003
			term 730
	set "hai start"
	# need to set it here to not have it trigger alongside the invisible mission
	set "First Contact: Hai: offered"
	set "Intro [0]: offered"
	set "skip FW"
	set "skip HR"

conversation "hai intro"
	scene "scene/hai shipyard"

> 	`	You've never been to the shipyard before, and as you arrive, it already feels like you're traveling to another world. The pictures and video you've seen of all these ships before simply don't do them justice; seeing them for real is something else entirely. Even though you know a lot of those ships must have been in service for many years, you are stunned by how new and shiny they look. You begin to wonder if you are ready to take the big leap.`

Reviewer 8: ```suggestion
	`	You've never been to the shipyard before, and as you arrive, it already feels like you're traveling to another world. The pictures and videos you've seen of all these ships before simply don't do them justice; seeing them for real is something else entirely. Even though you know a lot of those ships must have been in service for many years, you are stunned by how new and shiny they look. You begin to wonder if you are ready to take the big leap.`
```

	`	This "Aphid" could suit your needs. It is fast and able to carry a respectable amount of cargo.`
	scene "thumbnail/hai aphid"
	`	That is, unless you want to get adventurous and try out the impetuous Grasshopper. Surely the interceptor would be able to beat the smaller human warships, provided a competent pilot is at the helm.`

> 	`	You have, after all, heard tales of pirates and quick money... And up north you can find the 'Unfettered'. Maybe they could come to like you, if you were to show them the strength and cunning they respect.`

Reviewer 8: ```suggestion
	`	You have, after all, heard tales of pirates and quick money... And up north you can find the 'Unfettered'. Maybe they would come to like you, if you were to show them the strength and cunning they respect.`
```

	choice
		`	(Go to the bank.)`
	scene "scene/hai bank"
	`	You head to the bank's lobby, hoping to get enough of a loan to be able to buy one of these pristine ships.`

> 	`	To your disapointment the only ship you could afford is an Aphid. You arrange the contract to loan as much money as you need in order to buy one, in what has to be the first seriously unwise act of your life. You may have to sell some part of the ship in order to be able to afford flying it!`

Reviewer 8: ```suggestion
	`	To your disapointment the only ship you could afford is an Aphid. You arrange the contract to loan as much money as you need in order to buy one, in what has to be the first seriously unwise act of your life. You may have to sell some parts of the ship to afford flying it!`
```

	`	You can probably sell the shield regenerator without any consequences, so long as you do not get too close to hostile territory. You may even downgrade the anti-missile turret.`

> 	`	The bank employee sums it up to you, with a hesitant smile. "This will put you in a debt of 1 million 431 thousand credits, due in the course of 2 years, with an interest rate of 0.3% per day. Please sign here."`

Reviewer 8: > 1 million 431 thousand

This hurts my eyes

Author: ```suggestion
	`	The bank employee sums it up to you, with a hesitant smile. "This will put you in a debt of 1,431 thousand credits, due in the course of 2 years, with an interest rate of 0.3% per day. Please sign here."`
```
?

	name
	`	You head out of the bank, but on your way to buy the Aphid at the shipyard, you see a notice. A retiring captain is offering to sell some of his ships at an interesting price, including a Grasshopper... Maybe they could sell it to you.`
	choice
		`	(Go to the shipyard.)`
			decline
		`	(Go see what the captain has to offer first.)`
	`	You head to the spaceport and easily find the captain selling off his ships. Thankfully they did not yet sell the Grasshopper.`

> 	`	The captain greets you in Hai, but you've somehow managed to never really learn the language. Most everyone speaks yours to at least some degree anyway, and once they realize you do not understand, they switch with little effort.`

Reviewer 8: ```suggestion
	`	The captain greets you in Hai, but you've somehow managed to never really learn the language. Most people speaks yours to at least some degree anyway, and once they realize you do not understand them, they switch with little effort.`
```
This paragraph needs some more work.

Author: `	The captain greets you in Hai, but when you explain to him in your language that you never learned Hai, he switches to speaking relatively fluidly in your language.`

Reviewer 8: ```
`	The captain greets you in Hai. You are embarrassed to have to explain to them that you never learned their language. Most Hai speak your language however, and, as usual, they are able to switch to speaking yours without much difficulty.`
```
Still not sure.

	`	"Hello, good fellow! I assume you come for the Grasshopper." Your interest for this ship must be apparent.`
	choice
		`	"Yes indeed."`
		`	"How did you know?"`
		`	"Sorry, not interested."`
			decline

> 	`	"I can see the will to explore in your eyes. I would be delighted to have you flying the 'Pristine Jewel'!" they say, taking you to the ship. It is clearly not brand new but still looks in surprisingly good condition.`

Reviewer 8: ```suggestion
	`	"I can see the will to explore in your eyes. I would be delighted to have you flying the 'Pristine Jewel'!" they say, taking you to the ship. It is clearly not brand new, but is still in a surprisingly good condition.`
```

	`	The only moment when he becomes truly animated is when explaining the Truth in Lending statement to you. "You are borrowing 480,000 credits," he says, "to be repaid over the course of one year. Your daily interest rate is 0.4%, which means that your daily payments are 2,503 credits, and by the end of the year you will have paid... 434,000 credits in interest. Sign here." He grins.`
	`	You sign your name one final time:`
	name
	`	The elevator is so well-tuned that you do not even realize it is moving until it has deposited you back in the lobby. But as you leave the bank, you are smiling. This crazy adventure suddenly feels real to you. You are going to do it. You are finally going to get off this planet.`
	`	Compared to the bank, you feel much more at home in the shipyard, walking among the rusted out hulks and newer ships that gleam in the sunlight. You smell grease and dirt and rocket fuel; wonderful smells. There are three ship models within your price range. Which one you choose will determine your future...`

start "hai space after FW and HR"

> 	name "Advanced Hai Start"

Author: Far Forest Start?
To be fair, the Hai were Revealed already so that would fit well!

> 	name "Advanced Hai Start"

Author: ```suggestion
	name "Hai Legacy"
```
What about this?

	description `You grew up on Greenwater, a world with a very rich and mixed culture. You've dreamed of owning your own fleet since you were given toy spaceships as a small child. In order to reach that goal, you worked hard making human clothing that sold well.`
	description `	You were inspired by stories of heroes saving both humans and Hai.`
	description `	Now, as you enter the shipyard, you feel like this is the start of a new life, in space.`
	thumbnail "scene/hai start"
	system "Fah Soom"
	planet "Greenwater"
	date 18 6 3020
	to display
		or
			has "main plot completed"
			has "Ask the Hai about the Unfettered"
	on display
		description "What if you had not grown up on New Boston? What if you had waited longer before taking to the skies?"
		thumbnail ""
		date "middle of 3020"
		planet "Slightly too warm"
		credits "over a million"
	to reveal
		has "main plot completed"
		has "Ask the Hai about the Unfettered"
	on reveal

> 		name "Hai Start"

Reviewer 10: ![image](https://user-images.githubusercontent.com/4471575/217703295-3ea62468-fa0f-4572-8c29-054b30306280.png)

Looks like a clean fit, decent amount of room still.

	`	The only moment when he becomes truly animated is when explaining the Truth in Lending statement to you. "You are borrowing 480,000 credits," he says, "to be repaid over the course of one year. Your daily interest rate is 0.4%, which means that your daily payments are 2,503 credits, and by the end of the year you will have paid... 433,567 credits in interest. Sign here." He grins.`
	`	You sign your name one final time:`
	name
	`	The elevator is so well-tuned that you do not even realize it is moving until it has deposited you back in the lobby. But as you leave the bank, you are smiling. This crazy adventure suddenly feels real to you. You are going to do it. You are finally going to get off this planet.`
	`	Compared to the bank, you feel much more at home in the shipyard, walking among the rusted out hulks and newer ships that gleam in the sunlight. You smell grease and dirt and rocket fuel; wonderful smells. There are three ship models within your price range. Which one you choose will determine your future...`

start "hai space after FW and HR"
	name "Hai Legacy"

> 	description `You grew up on Greenwater, a world with a very rich and mixed culture. You've dreamed of owning your own fleet since you were given toy spaceships as a small child. In order to reach that goal, you worked hard making human clothing that sold well.`

Reviewer 7: One thing that occurs to me here, is that either earning that much or getting a loan for that much would be comparatively difficult compared to the New Boston start.
It could be possible that you did some sort of competition or aptitude test and received a grant as a result that could give you a leg up. That way you might start with a smaller loan (maybe only like 800k?) but still have the same amount of cash for the start.
It might be a little more believable than getting to own a ship from working in artisanal retail.

Author: This start is easier than the new boston start, no doubt about it, you have way more cargo capacity and better jobs available
if anarchist keeps on nerfing Hai space it'll be a different thing, but rn it's not issue

Author: the way I see it, Hai currency is different and the risks are really lower - there arent pirates everywhere, just unfettered if you dont know about wah ki but that's easy to avoid
oh also the % to pay is lower here iirc

	choice
		`	"Sold!"`
		`	"Sorry, not interested."`
			decline
	`	As the two of you settle up the paperwork, the captain takes the time to explain, in great detail, why this ship is so marvelous. Because of how excited you are to finally fly, you miss everything other than repeated reference to the 'exceptional' reinforced armor.`
	`	Before you leave, they give you a final tip: "If you want to, you can always sell this ship and buy an Aphid instead. The shipyard authorities will give you a discount for having sold my ship."`
	action
		set "hai start: Grasshopper"
		give ship "Grasshopper (Expensive)" "Pristine Jewel"
		payment -1132000

# this ship uses the most expensive loadout possible (3.5M)
# so that you +- have the same amount of value back if you sell it (1.3mill)
# It has a small hull buff to help the player survive.
ship "Grasshopper" "Grasshopper (Expensive)"
	add attributes
		"hull" 400
	outfits
		`"Benga" Atomic Thruster`
		`"Biroo" Atomic Steering`
		"Fuel Pod"
		"Hai Chasm Batteries"
		"Hai Corundum Regenerator"
		"Hai Williwaw Cooling"
		Hyperdrive
		"Ion Cannon"
		"Pebble Core"
		"Pulse Rifle"

> 		"Quantum Keystone" 2

Reviewer 11: Why does it have two Keystones?

Author: Because you look like you need extra luck!
Or mb it was for himself, who knows (but its intended)
~~mb it just fits perfectly this way~~

Reviewer 11: OK. (you can resolve this)

start "hai space after FW and HR"
	name "Advanced Hai Start"
	description `You grew up on Greenwater, a world with a very rich and diverse hai-human culture. You've dreamed of owning your own fleet since you were given toy spaceships as a small child. In order to reach that goal, you worked hard in a texture farm making human clothing that sold very well.`

> 	description `	You heard stories of heroes saving both humans and Hai.`

Reviewer 8: Okay, now we just need a reason why this fact is mentioned. I know why _you_ want it there, but it doesn't really make sense for FL; you first talk about FL's life, then this random story bit, then back to what FL's doing rn. At least now it doesn't imply that FL and these events are somehow connected, so that's fixed.

Author: its mentionned because thats part of why you take to the stars
```suggestion
	description `	You were inspired by stories of heroes saving both humans and Hai.`
```

start "hai space"
	name "Hai Origins"
	description `You grew up on Greenwater, a world with a rich and mixed culture. You've dreamed of owning your own fleet since you were given toy spaceships as a small child. In order to reach that goal, you worked hard at a travel company, recommending locations in human space.`
	description `	Now, as your ferry docks at the main spaceport, you feel like this is the start of a new life as a captain.`
	thumbnail "scene/hai start"
	system "Fah Soom"
	planet "Greenwater"
	date 16 11 3013
	to reveal

> 		has "Ask the Hai about the Unfettered"

Reviewer 3: ```suggestion
		has "Ask the Hai about the Unfettered: offered"
```

	on reveal
		name "Hai Origins"
		description `You grew up on Greenwater, in one of the human cities built on the equator, where the climate is too hot for the Hai. You've dreamed of owning your own fleet since you were given toy spaceships as a small child. In order to reach that goal, you worked hard at a travel company, recommending locations in human space.`
		description `	Now, as your ferry docks at the main spaceport, you feel like this is the start of a new life as a captain.`
		system "Fah Soom"
		planet "Greenwater"
		date 16 11 3013
		credits 1431000
		debt 1431000
	to unlock
		has "main plot completed"

> 		has "Ask the Hai about the Unfettered"

Reviewer 3: ```suggestion
		has "Ask the Hai about the Unfettered: offered"
```

	on display
		description "What if you had not grown up on New Boston? What if you had waited longer before taking to the skies?"
		thumbnail ""
		date "3020"
		planet "Slightly too warm"
		credits "over a million"
	to reveal
		has "main plot completed"
		has "Ask the Hai about the Unfettered"
	on reveal
		name "Undiscovered Hai Start"
		description "There could be interesting possibilities up north, once all crises are resolved."

> 		description "	You could have started a different way, went a different path."

Reviewer 8: Are you sure about that tab?

Author: the existing start has this as well between lines?

Reviewer 8: Oops. I thought it was placed after the deleted lines, where it would have been the first description line. My bad.

	name
	`	You head out of the bank, but on your way to buy the Aphid at the shipyard, you see a notice. A retiring captain is offering to sell some of his ships at an interesting price, including a Grasshopper... Maybe they could sell it to you.`
	choice
		`	(Go to the shipyard.)`
			decline
		`	(Go see what the captain has to offer first.)`
	`	You head to the spaceport and easily find the captain selling off his ships. Thankfully they did not yet sell the Grasshopper.`
	`	The captain greets you in Hai. You are embarrassed to have to explain to them that you never learned it, due to most Hai speaking your language. As usual, they are able to switch to speaking yours without much difficulties.`
	`	"Hello, good fellow! I assume you come for the Grasshopper." Your interest for this ship must be apparent.`
	choice
		`	"Yes indeed."`
		`	"How did you know?"`
		`	"Sorry, not interested."`
			decline
	`	"I can see the will to explore in your eyes. I would be delighted to have you flying the 'Pristine Jewel'!" they say, taking you to the ship. It is clearly not brand new, but is still in a surprisingly good condition.`
	scene "thumbnail/hai grasshopper"

> 	`	"Because I like you I'll give you a friendly price. What do you say about... 1.132M credits, for this wonderful ship? That's less than a third the price you would have to pay if you were to buy this model from the shipyard. Except you can't find a model like this in shipyards, trust me!"`

Reviewer 8: Honestly, I prefer the first one, but if you feel like we have to put the exact amount there, this also works. I just don't see anyone actually telling you `what about... 1.132` like it's a number they came up with on the spot. (Also, I'm changing M to million because that's how people say it in the game; M is only used in the outfitter/shipyard iirc.)
```suggestion
	`	"Because I like you I'll give you a friendly price. What do you say about... 1.132 million credits, for this wonderful ship? That's less than a third the price you would have to pay if you were to buy this model from the shipyard. Except you can't find a model like this in shipyards, trust me!"`
```

Author: It used to be 1 M I now changed it to be more caus otherwise its too easy
```suggestion
	`	"Because I like you I'll give you a friendly price. What do you say about... a bit over a million credits, for this wonderful ship? That's less than a third the price you would have to pay if you were to buy this model from the shipyard. Except you can't find a model like this in shipyards, trust me!"`
```
what about this?

Reviewer 8: Works I guess.

	choice
		`	"Sold!"`
		`	"Sorry, not interested."`
			decline
	`	As the two of you settle up the paperwork, the captain takes the time to explain, in great detail, why this ship is so marvelous. Because of how excited you are to finally fly, you miss everything other than repeated reference to the 'exceptional' reinforced armor.`

> 	`	Before you leave, they give you a final tip: "If you want, you can always sell this ship and buy an Aphid instead, as the shipyard authorities will make you a discount for having sold my ship."`

Reviewer 8: ```suggestion
	`	Before you leave, they give you a final tip: "If you want to, you can always sell this ship and buy an Aphid instead. The shipyard authorities will give you a discount for having sold my ship."`
```

conversation "hai intro"
	scene "scene/hai shipyard"

> 	`	You've never been to the shipyard before, and going to the poles already feels like you're traveling to another world. The pictures and videos you've seen of all these ships before simply don't do them justice; seeing them for real is something else entirely. Even though you know a lot of those ships must have been in service for many years, you are stunned by how new and shiny they look. You begin to wonder if you are ready to take the big leap.`

Reviewer 8: Honestly I prefer the previous version. Also, it feels weird since nobody said a word about the shipyard being near the poles.

Author: ![image](https://user-images.githubusercontent.com/94366726/214269370-e0a16a4d-c820-418b-8357-240140feee00.png)

Reviewer 8: FL in _this_ start has never read that description.

Reviewer 10: The player doesn't lose information doing this new start, though...? It's taking information that's true about the world and extrapolating an interesting concept from it. I don't understand where this part of the comment is coming from.

Reviewer: The phrase, "going to the poles", would not make much sense to a player. Could you please choose "North Pole" or "South Pole"?

	`	Making your way towards the shipyard, you pull up the human-language version of your loan agreement to review: You have borrowed 1.431 million credits to be repaid over the course of two years. Your interest rate is 0.3%, making your repayments 4,836 credits per day.`
	`	You look up from your tablet just before running head-on into a gray-furred Hai wearing a flight suit, knocking him over. As you bend over to help, he takes a glance at your mortgage documents. "You're about to become a captain, aren't you?" he says after dusting himself off.`
	
	choice
		`	"Yes, I am."`
		`	"How could you tell?"`
			goto tell
		`	"Who said you could look at that?"`
			goto look
	
	`	"Then maybe I can help. Follow me."`
		goto help
	
	label tell
	`	"From your loan," he replies. "There's only one reason to have that exact number of credits. Speaking of which, I think I can help you."`
		goto help
	
	label look
	`	He scratches the back of his head. "Sorry. I was just trying to help, but if I'm causing too much trouble, I'll get out of your way.`
	choice
		`	"I don't mind."`

> 				goto happy

Reviewer 3: ```suggestion
			goto happy
```

	
	label help
	`	The Hai brings you to the landing pads of Greenwater, where a large number of humans and Hai are gathered around several vessels. From the crowd, another Hai approaches your companion, talking to him in Hai at such a speed that you'd have trouble understanding even if you knew the language. However, the gray-haired Hai simply nods and hands the other Hai a set of keys in exchange for a credit chip. The other Hai quickly bows before dashing towards one of the landed ships.`
	`	"I'm selling off my fleet before retiring," the Hai says. "Sure, I could just go to the shipyard, but I could never have started if it weren't for another retiring captain selling me a ship for cheap when I was young. So, I've decided to pay it forward and help out other captains-to-be."`
	`	The Hai walks up to one of the smaller ships in the fleet, a Grasshopper interceptor. "The Pristine Jewel. She's served me well over the past century, but I'm sure she'll be safe in your hands."`
	choice
		`	"You're giving it to me?"`
		`	"Is there a freighter I could take instead?"`
			goto freighter
		`	"What about the other people here? Shouldn't they get first pick?"`
			goto pick
	

> 	`	The Hai chuckles. "That would be nice, wouldn't it? Unfortunately, retirement ain't cheap. I'd be happy to part with her for 1.132 million credits, though; that's less than a Aphid, let alone a new Grasshopper.`

Reviewer 3: ```suggestion
	`	The Hai chuckles. "That would be nice, wouldn't it? Unfortunately, retirement isn't cheap. I'd be happy to part with her for 1.132 million credits, though; that's less than a Aphid, let alone a new Grasshopper.`
```

		goto deal
	
	label freighter
	`	"Unfortunately, all my Aphids were bought earlier today. Still, I'm sure you can survive; the Unfettered'll be a tough ask, but pirates in human space won't be much of a match. Plus, if you still can't handle it, you can always sell her. She'll only cost you 1.132 million credits, and you'll be able to use the old parts of the Pristine Jewel to buy an Aphid at a discount.`
		goto deal
	
	label pick
	`	The Hai shakes his head. "Don't worry. Most of the crowd are just spectators. Usually it'd be a human captain pulling a stunt like selling their fleet publicly; most retiring Hai just donate their fleets to the government. I'm not exactly handing these ships out for free either. If you want her, the Pristine Jewel's 1.132 million credits.`
	
	label deal
	`	"So, do we have a deal?"`
	choice
		`	"We do!"`
		`	"I think I'd be better off buying from the shipyard."`
			goto leave
	
	`	"Great!" He holds out a data tablet. "Just sign here, and she's yours."`
	name
	action
		set "hai space start: Grasshopper"
		give ship "Grasshopper (Hai Start)" "Pristine Jewel"
		payment -1132000

> 	`	The Hai gives you the keys to the Grasshopper after you hand over the tablet and your payment. "Looks like this your first step to becoming an interstellar entrepreneur. I'd say we'll meet again someday, but I'd rather not lie. In all my years, I never did meet the captain who sold me my first ship again. Not that should matter, anyway; this is your story, not mine."`

Reviewer 3: ```suggestion
	`	The Hai gives you the keys to the Grasshopper after you hand over the tablet and your payment. "Looks like this your first step to becoming an interstellar entrepreneur. I'd say we'll meet again someday, but I'd rather not lie. In all my years, I never did meet the captain who sold me my first ship again. Not that that should matter, anyway; this is your story, not mine."`
```
